diff -Naur cdrecord-1.8.1/cdrecord/cdrecord.1 cdrecord-1.8.1-p/cdrecord/cdrecord.1
--- cdrecord-1.8.1/cdrecord/cdrecord.1	Thu Apr 27 14:06:43 2000
+++ cdrecord-1.8.1-p/cdrecord/cdrecord.1	Thu May 25 19:17:08 2000
@@ -118,7 +118,7 @@
 described below.
 
 .PP
-If a file /etc/default/cdrecord exists, the parameter to the
+If a file /etc/cdrecord.conf exists, the parameter to the
 .B dev=
 option may also be a drive name label in said file (see FILES section).
 
@@ -445,7 +445,7 @@
 .B dev=
 option does not contain the characters ',', '/', '@' or ':',
 it is interpreted as an label name that may be found in the file
-/etc/default/cdrecord (see FILES section).
+/etc/cdrecord.conf (see FILES section).
 .TP
 .BI timeout= #
 Set the default SCSI command timeout value to 
@@ -957,7 +957,7 @@
 .TP
 CDR_DEVICE
 This may either hold a device identifier that is suitable to the open
-call of the SCSI transport library or a label in the file /etc/default/cdrecord.
+call of the SCSI transport library or a label in the file /etc/cdrecord.conf.
 .TP
 CDR_SPEED
 Sets the default speed value for writing (see also 
@@ -971,8 +971,8 @@
 
 .SH FILES
 .TP
-/etc/default/cdrecord
-Default values can be set for the following options in /etc/default/cdrecord.
+/etc/cdrecord.conf
+Default values can be set for the following options in /etc/cdrecord.conf
 For example:
 .SM CDR_FIFOSIZE=8m
 or
@@ -981,7 +981,7 @@
 .TP
 CDR_DEVICE
 This may either hold a device identifier that is suitable to the open
-call of the SCSI transport library or a label in the file /etc/default/cdrecord 
+call of the SCSI transport library or a label in the file /etc/cdrecord.conf
 that allows to identify a specific drive on the system.
 .TP
 CDR_SPEED
diff -Naur cdrecord-1.8.1/cdrecord/cdrecord.dfl cdrecord-1.8.1-p/cdrecord/cdrecord.dfl
--- cdrecord-1.8.1/cdrecord/cdrecord.dfl	Sun Apr 16 02:08:15 2000
+++ cdrecord-1.8.1-p/cdrecord/cdrecord.dfl	Thu May 25 19:17:08 2000
@@ -1,6 +1,6 @@
 #ident @(#)cdrecord.dfl	1.2 00/04/16 Copyr 1998 J. Schilling
 #
-# This file is /etc/default/cdrecord
+# This file is /etc/cdrecord.conf
 # It contains defaults that are used if no command line option
 # or environment is present.
 # 
diff -Naur cdrecord-1.8.1/cdrecord/defaults.c cdrecord-1.8.1-p/cdrecord/defaults.c
--- cdrecord-1.8.1/cdrecord/defaults.c	Sun Apr 16 02:08:58 2000
+++ cdrecord-1.8.1-p/cdrecord/defaults.c	Thu May 25 19:17:08 2000
@@ -32,6 +32,8 @@
 #include <utypes.h>
 #include "cdrecord.h"
 
+#define DEFAULT_CONFIG	"/etc/cdrecord.conf"
+
 EXPORT	void	cdr_defaults	__PR((char **devp, int *speedp, long *fsp));
 LOCAL	void	cdr_xdefaults	__PR((char **devp, int *speedp, long *fsp));
 LOCAL	char *	strsv		__PR((char* s));
@@ -49,7 +51,7 @@
 	if (!dev) {
 		*devp = getenv("CDR_DEVICE");
 
-		if (!*devp && defltopen("/etc/default/cdrecord") == 0) {
+		if (!*devp && defltopen(DEFAULT_CONFIG) == 0) {
 			dev = defltread("CDR_DEVICE=");
 			if (dev != NULL)
 				*devp = strsv(dev);
@@ -62,7 +64,7 @@
 		char	*p = getenv("CDR_SPEED");
 
 		if (!p) {
-			if (defltopen("/etc/default/cdrecord") == 0) {
+			if (defltopen(DEFAULT_CONFIG) == 0) {
 				p = defltread("CDR_SPEED=");
 			}
 		}
@@ -79,7 +81,7 @@
 		char	*p = getenv("CDR_FIFOSIZE");
 
 		if (!p) {
-			if (defltopen("/etc/default/cdrecord") == 0) {
+			if (defltopen(DEFAULT_CONFIG) == 0) {
 				p = defltread("CDR_FIFOSIZE=");
 			}
 		}
@@ -110,7 +112,7 @@
 		x++;
 	}
 	sprintf(dname, "%s=", p);
-	if (defltopen("/etc/default/cdrecord") != 0)
+	if (defltopen(DEFAULT_CONFIG) != 0)
 		return;
 
 	p = defltread(dname);
diff -Naur cdrecord-1.8.1/doc/cdrecord.man cdrecord-1.8.1-p/doc/cdrecord.man
--- cdrecord-1.8.1/doc/cdrecord.man	Thu Apr 27 14:07:33 2000
+++ cdrecord-1.8.1-p/doc/cdrecord.man	Thu May 25 19:17:08 2000
@@ -50,7 +50,7 @@
      the ----ssssccccaaaannnnbbbbuuuussss option of ccccddddrrrreeeeccccoooorrrrdddd described below.
 
 
-     If a file /etc/default/cdrecord exists, the parameter to the
+     If a file /etc/cdrecord.conf exists, the parameter to the
      ddddeeeevvvv==== option may also be a drive name label in said file (see
      FILES section).
 
@@ -367,7 +367,7 @@
           If the argument to the ddddeeeevvvv==== option does not contain the
           characters  ',',  '/', '@' or ':', it is interpreted as
           an  label  name  that  may  be  found   in   the   file
-          /etc/default/cdrecord (see FILES section).
+          /etc/cdrecord.conf (see FILES section).
 
      ttttiiiimmmmeeeeoooouuuutttt====#
           Set the default SCSI command timeout value to  #.   The
@@ -888,7 +888,7 @@
      CDR_DEVICE
           This may either hold a device identifier that is  suit-
           able  to the open call of the SCSI transport library or
-          a label in the file /etc/default/cdrecord.
+          a label in the file /etc/cdrecord.conf.
 
      CDR_SPEED
           Sets the default speed value for writing (see  also   ----
@@ -900,16 +900,16 @@
 
 
 FFFFIIIILLLLEEEESSSS
-     /etc/default/cdrecord
+     /etc/cdrecord.conf
           Default values can be set for the following options  in
-          /etc/default/cdrecord.   For  example:  CDR_FIFOSIZE=8m
+          /etc/cdrecord.conf.   For  example:  CDR_FIFOSIZE=8m
           or CDR_SPEED=2
 
           CDR_DEVICE
                This may either hold a device identifier  that  is
                suitable  to  the  open call of the SCSI transport
                library    or    a    label    in     the     file
-               /etc/default/cdrecord  that  allows  to identify a
+               /etc/cdrecord.conf  that  allows  to identify a
                specific drive on the system.
 
           CDR_SPEED
diff -Naur cdrecord-1.8.1/include/deflts.h cdrecord-1.8.1-p/include/deflts.h
--- cdrecord-1.8.1/include/deflts.h	Sat Dec  5 14:04:18 1998
+++ cdrecord-1.8.1-p/include/deflts.h	Thu May 25 19:17:08 2000
@@ -27,7 +27,7 @@
 #include <mconfig.h>
 #endif
 
-#define	DEFLT	"/etc/default"
+#define	DEFLT	"/etc"
 
 /*
  * cmd's to defltcntl()
