--- cdrtools-1.10/conf/aclocal.m4.orig	Thu Aug  2 23:54:08 2001
+++ cdrtools-1.10/conf/aclocal.m4	Fri Aug  3 00:00:43 2001
@@ -1331,96 +1331,6 @@
 
 #serial 18
 
-dnl By default, many hosts won't let programs access large files;
-dnl one must use special compiler options to get large-file access to work.
-dnl For more details about this brain damage please see:
-dnl http://www.sas.com/standards/large.file/x_open.20Mar96.html
-
-dnl Written by Paul Eggert <eggert@twinsun.com>.
-
-dnl Internal subroutine of AC_SYS_LARGEFILE.
-dnl AC_SYS_LARGEFILE_TEST_INCLUDES
-AC_DEFUN(AC_SYS_LARGEFILE_TEST_INCLUDES,
-  [[#include <sys/types.h>
-    /* Check that off_t can represent 2**63 - 1 correctly.
-       We can't simply "#define LARGE_OFF_T 9223372036854775807",
-       since some C++ compilers masquerading as C compilers
-       incorrectly reject 9223372036854775807.  */
-#   define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
-    int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
-			&& LARGE_OFF_T % 2147483647 == 1)
-		       ? 1 : -1];
-  ]])
-
-dnl Internal subroutine of AC_SYS_LARGEFILE.
-dnl AC_SYS_LARGEFILE_MACRO_VALUE(C-MACRO, VALUE, CACHE-VAR, COMMENT, INCLUDES, FUNCTION-BODY)
-AC_DEFUN(AC_SYS_LARGEFILE_MACRO_VALUE,
-  [AC_CACHE_CHECK([for $1 value needed for large files], $3,
-     [$3=no
-      AC_TRY_COMPILE([$5],
-	[$6], 
-	,
-	[AC_TRY_COMPILE([#define $1 $2]
-[$5]
-	   ,
-	   [$6],
-	   [$3=$2])])])
-   if test "[$]$3" != no; then
-     AC_DEFINE_UNQUOTED([$1], [$]$3, [$4])
-   fi])
-
-AC_DEFUN(AC_SYS_LARGEFILE,
-  [AC_ARG_ENABLE(largefile,
-     [  --disable-largefile     omit support for large files])
-   if test "$enable_largefile" != no; then
-
-     AC_CACHE_CHECK([for special C compiler options needed for large files],
-       ac_cv_sys_largefile_CC,
-       [ac_cv_sys_largefile_CC=no
-        if test "$GCC" != yes; then
-	  # IRIX 6.2 and later do not support large files by default,
-	  # so use the C compiler's -n32 option if that helps.
-	  AC_TRY_COMPILE(AC_SYS_LARGEFILE_TEST_INCLUDES, , ,
-	    [ac_save_CC="${CC-cc}"
-	     CC="$CC -n32"
-	     AC_TRY_COMPILE(AC_SYS_LARGEFILE_TEST_INCLUDES, ,
-	       ac_cv_sys_largefile_CC=' -n32')
-	     CC="$ac_save_CC"])
-        fi])
-     if test "$ac_cv_sys_largefile_CC" != no; then
-       CC="$CC$ac_cv_sys_largefile_CC"
-     fi
-
-     AC_SYS_LARGEFILE_MACRO_VALUE(_FILE_OFFSET_BITS, 64,
-       ac_cv_sys_file_offset_bits,
-       [Number of bits in a file offset, on hosts where this is settable.],
-       AC_SYS_LARGEFILE_TEST_INCLUDES)
-     AC_SYS_LARGEFILE_MACRO_VALUE(_LARGE_FILES, 1,
-       ac_cv_sys_large_files,
-       [Define for large files, on AIX-style hosts.],
-       AC_SYS_LARGEFILE_TEST_INCLUDES)
-   fi
-  ])
-
-
-AC_DEFUN(AC_FUNC_FSEEKO,
-  [AC_SYS_LARGEFILE_MACRO_VALUE(_LARGEFILE_SOURCE, 1,
-     ac_cv_sys_largefile_source,
-     [Define to make fseeko visible on some hosts (e.g. glibc 2.2).],
-     [#include <stdio.h>], [return !fseeko;])
-   # We used to try defining _XOPEN_SOURCE=500 too, to work around a bug
-   # in glibc 2.1.3, but that breaks too many other things.
-   # If you want fseeko and ftello with glibc, upgrade to a fixed glibc.
-
-   AC_CACHE_CHECK([for fseeko], ac_cv_func_fseeko,
-     [ac_cv_func_fseeko=no
-      AC_TRY_LINK([#include <stdio.h>],
-        [return fseeko && fseeko (stdin, 0, 0);],
-	[ac_cv_func_fseeko=yes])])
-   if test $ac_cv_func_fseeko != no; then
-     AC_DEFINE(HAVE_FSEEKO, 1,
-       [Define if fseeko (and presumably ftello) exists and is declared.])
-   fi])
 
 
 dnl XXXXXXXXXXXXXXXXXX End Stolen from GNU tar XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
--- cdrtools-1.10/conf/xconfig.h.in.orig	Thu Apr 12 18:10:41 2001
+++ cdrtools-1.10/conf/xconfig.h.in	Fri Aug  3 11:31:38 2001
@@ -8,79 +8,79 @@
 /*
  * Header Files
  */
-#undef PROTOTYPES	/* if Compiler supports ANSI C prototypes */
-#undef HAVE_STDARG_H	/* to use stdarg.h, else use varargs.h NOTE: SaberC on a Sun has prototypes but no stdarg.h */
-#undef HAVE_VARARGS_H	/* to use use varargs.h NOTE: The free HP-UX C-compiler has stdarg.h but no PROTOTYPES */
-#undef HAVE_STDLIB_H	/* to use general utility defines (malloc(), size_t ...) and general C library prototypes */
-#undef HAVE_STRING_H	/* to get NULL and ANSI C string function prototypes */
-#undef HAVE_STRINGS_H	/* to get BSD string function prototypes */
-#undef STDC_HEADERS	/* if ANSI compliant stdlib.h, stdarg.h, string.h, float.h are present */
-#undef HAVE_UNISTD_H	/* to get POSIX syscall prototypes XXX sys/file.h fcntl.h (unixstd/fctl)XXX*/
-#undef HAVE_GETOPT_H	/* to get getopt() prototype from getopt.h instead of unistd.h */
-#undef HAVE_LIMITS_H	/* to get POSIX numeric limits constants */
-#undef HAVE_A_OUT_H	/* if a.out.h is present (may be a system using a.out format) */
-#undef HAVE_AOUTHDR_H	/* if aouthdr.h is present. This is a COFF system */
-#undef HAVE_ELF_H	/* if elf.h is present. This is an ELF system */
-#undef HAVE_FCNTL_H	/* to access, O_XXX constants for open(), otherwise use sys/file.h */
-#undef HAVE_SYS_FILE_H	/* to use O_XXX constants for open() and flock() defs */
-#undef HAVE_INTTYPES_H	/* to use UNIX-98 inttypes.h */
-#undef HAVE_DIRENT_H	/* to use POSIX dirent.h */
-#undef HAVE_SYS_DIR_H	/* to use BSD sys/dir.h */
-#undef HAVE_NDIR_H	/* to use ndir.h */
-#undef HAVE_SYS_NDIR_H	/* to use sys/ndir.h */
-#undef HAVE_MALLOC_H	/* if malloc.h exists */
-#undef HAVE_TERMIOS_H	/* to use POSIX termios.h */
-#undef HAVE_TERMIO_H	/* to use SVR4 termio.h */
-#undef HAVE_SHADOW_H	/* if shadow.h exists */
-#undef HAVE_SYSLOG_H	/* if syslog.h exists */
-#undef HAVE_SYS_TIME_H	/* may include sys/time.h for struct timeval */
-#undef TIME_WITH_SYS_TIME	/* may include both time.h and sys/time.h */
-#undef HAVE_TIMES	/* to use times() and sys/times.h */
-#undef HAVE_SYS_TIMES_H	/* may include sys/times.h for struct tms */
-#undef HAVE_UTIMES		/* to use BSD utimes() and sys/time.h */
-#undef HAVE_UTIME_H		/* to use utime.h for the utimbuf structure declaration, else declare struct utimbuf yourself */
-#undef HAVE_SYS_UTIME_H		/* to use sys/utime.h if utime.h does not exist */
-#undef HAVE_SYS_IOCTL_H		/* if sys/ioctl.h is present */
-#undef HAVE_SYS_FILIO_H		/* if sys/ioctl.h is present */
-#undef HAVE_SYS_PARAM_H		/* if sys/param.h is present */
-#undef HAVE_MNTENT_H		/* if mntent.h is present */
-#undef HAVE_SYS_MNTENT_H	/* if sys/mntent.h is present */
-#undef HAVE_SYS_MNTTAB_H	/* if sys/mnttab.h is present */
-#undef HAVE_SYS_MOUNT_H		/* if sys/mount.h is present */
-#undef HAVE_WAIT_H		/* to use wait.h for prototypes and union wait */
-#undef HAVE_SYS_WAIT_H		/* else use sys/wait.h */
-#undef HAVE_SYS_RESOURCE_H	/* to use sys/resource.h for rlimit() and wait3() */
-#undef HAVE_SYS_PROCFS_H	/* to use sys/procfs.h for wait3() emulation */
-#undef HAVE_SYS_SYSTEMINFO_H	/* to use SVr4 sysinfo() */
-#undef HAVE_SYS_UTSNAME_H	/* to use uname() */
-#undef HAVE_SYS_PRIOCNTL_H	/* to use SVr4 priocntl() instead of nice()/setpriority() */
-#undef HAVE_SYS_RTPRIOCNTL_H	/* if the system supports SVr4 real time classes */
-#undef HAVE_SYS_MTIO_H		/* to use mtio definitions from sys/mtio.h */
-#undef HAVE_SYS_MMAN_H		/* to use definitions for mmap()/madvise()... from sys/mman.h */
-#undef HAVE_SYS_SHM_H		/* to use definitions for shmget() ... from sys/shm.h */
-#undef HAVE_SYS_IPC_H		/* to use definitions for shmget() ... from sys/ipc.h */
-#undef MAJOR_IN_MKDEV		/* if we should include sys/mkdev.h to get major()/minor()/makedev() */
-#undef MAJOR_IN_SYSMACROS	/* if we should include sys/sysmacros.h to get major()/minor()/makedev() */
-#undef HAVE_SYS_DKIO_H		/* if we may include sys/dkio.h for disk ioctls */
-#undef HAVE_SYS_DKLABEL_H	/* if we may include sys/dklabel.h for disk label */
-#undef HAVE_SUN_DKIO_H		/* if we may include sun/dkio.h for disk ioctls */
-#undef HAVE_SUN_DKLABEL_H	/* if we may include sun/dklabel.h for disk label */
-#undef HAVE_POLL_H		/* if we may include poll.h to use poll() */
-#undef HAVE_SYS_POLL_H		/* if we may include sys/poll.h to use poll() */
-#undef HAVE_SYS_SELECT_H	/* if we may have sys/select.h nonstandard use for select() on some systems*/
-#undef NEED_SYS_SELECT_H	/* if we need sys/select.h to use select() (this is nonstandard) */
-#undef HAVE_LINUX_PG_H		/* if we may include linux/pg.h for PP ATAPI sypport */
-#undef HAVE_CAMLIB_H		/* if we may include camlib.h for CAM SCSI transport definitions */
-#undef HAVE_IEEEFP_H		/* if we may include ieeefp.h for finite()/isnand() */
-#undef HAVE_FP_H		/* if we may include fp.h for FINITE()/IS_INF()/IS_NAN() */
-#undef HAVE_VALUES_H		/* if we may include values.h for MAXFLOAT */
-#undef HAVE_FLOAT_H		/* if we may include float.h for FLT_MAX */
-#undef HAVE__FILBUF		/* if we have _filbuf() for USG derived STDIO */
-#undef HAVE___FILBUF		/* if we have __filbuf() for USG derived STDIO */
-#undef HAVE_USG_STDIO		/* if we have USG derived STDIO */
-#undef HAVE_ERRNO_DEF		/* if we have errno definition in <errno.h> */
-#undef HAVE_VFORK_H		/* if we should include vfork.h for vfork() definitions */
-#undef HAVE_BSD_DEV_SCSIREG_H	/* if we have a NeXT Step compatible sg driver */
+#undef PROTOTYPES
+#undef HAVE_STDARG_H
+#undef HAVE_VARARGS_H
+#undef HAVE_STDLIB_H
+#undef HAVE_STRING_H
+#undef HAVE_STRINGS_H
+#undef STDC_HEADERS
+#undef HAVE_UNISTD_H
+#undef HAVE_GETOPT_H
+#undef HAVE_LIMITS_H
+#undef HAVE_A_OUT_H
+#undef HAVE_AOUTHDR_H
+#undef HAVE_ELF_H
+#undef HAVE_FCNTL_H
+#undef HAVE_SYS_FILE_H
+#undef HAVE_INTTYPES_H
+#undef HAVE_DIRENT_H
+#undef HAVE_SYS_DIR_H
+#undef HAVE_NDIR_H
+#undef HAVE_SYS_NDIR_H
+#undef HAVE_MALLOC_H
+#undef HAVE_TERMIOS_H
+#undef HAVE_TERMIO_H
+#undef HAVE_SHADOW_H
+#undef HAVE_SYSLOG_H
+#undef HAVE_SYS_TIME_H
+#undef TIME_WITH_SYS_TIME
+#undef HAVE_TIMES
+#undef HAVE_SYS_TIMES_H
+#undef HAVE_UTIMES
+#undef HAVE_UTIME_H
+#undef HAVE_SYS_UTIME_H
+#undef HAVE_SYS_IOCTL_H
+#undef HAVE_SYS_FILIO_H
+#undef HAVE_SYS_PARAM_H
+#undef HAVE_MNTENT_H
+#undef HAVE_SYS_MNTENT_H
+#undef HAVE_SYS_MNTTAB_H
+#undef HAVE_SYS_MOUNT_H
+#undef HAVE_WAIT_H
+#undef HAVE_SYS_WAIT_H
+#undef HAVE_SYS_RESOURCE_H
+#undef HAVE_SYS_PROCFS_H
+#undef HAVE_SYS_SYSTEMINFO_H
+#undef HAVE_SYS_UTSNAME_H
+#undef HAVE_SYS_PRIOCNTL_H
+#undef HAVE_SYS_RTPRIOCNTL_H
+#undef HAVE_SYS_MTIO_H
+#undef HAVE_SYS_MMAN_H
+#undef HAVE_SYS_SHM_H
+#undef HAVE_SYS_IPC_H
+#undef MAJOR_IN_MKDEV
+#undef MAJOR_IN_SYSMACROS
+#undef HAVE_SYS_DKIO_H
+#undef HAVE_SYS_DKLABEL_H
+#undef HAVE_SUN_DKIO_H
+#undef HAVE_SUN_DKLABEL_H
+#undef HAVE_POLL_H
+#undef HAVE_SYS_POLL_H
+#undef HAVE_SYS_SELECT_H
+#undef NEED_SYS_SELECT_H
+#undef HAVE_LINUX_PG_H
+#undef HAVE_CAMLIB_H
+#undef HAVE_IEEEFP_H
+#undef HAVE_FP_H
+#undef HAVE_VALUES_H
+#undef HAVE_FLOAT_H
+#undef HAVE__FILBUF
+#undef HAVE___FILBUF
+#undef HAVE_USG_STDIO
+#undef HAVE_ERRNO_DEF
+#undef HAVE_VFORK_H
+#undef HAVE_BSD_DEV_SCSIREG_H
 
 /*
  * Convert to SCHILY name
@@ -106,99 +106,99 @@
 /*
  * Library Functions
  */
-#undef HAVE_STRERROR		/* strerror() is present in libc */
-#undef HAVE_MEMMOVE		/* memmove() is present in libc */
-#undef HAVE_MLOCKALL		/* working mlockall() is present in libc */
-#undef HAVE_MMAP		/* working mmap() is present in libc */
-#undef _MMAP_WITH_SIZEP		/* mmap() needs address of size parameter */
-#undef HAVE_FLOCK		/* *BSD flock() is present in libc */
-#undef HAVE_LOCKF		/* lockf() is present in libc (XOPEN) */
-#undef HAVE_FCNTL_LOCKF		/* file locking via fcntl() is present in libc */
-#undef HAVE_FCHDIR		/* fchdir() is present in libc */
-#undef HAVE_STATVFS		/* statvfs() is present in libc */
-#undef HAVE_QUOTACTL		/* quotactl() is present in libc */
-#undef HAVE_QUOTAIOCTL		/* use ioctl(f, Q_QUOTACTL, &q) instead of quotactl() */
-#undef HAVE_SETREUID		/* setreuid() is present in libc */
-#undef HAVE_SETRESUID		/* setresuid() is present in libc */
-#undef HAVE_SETEUID		/* seteuid() is present in libc */
-#undef HAVE_SETUID		/* setuid() is present in libc */
-#undef HAVE_SETREGID		/* setregid() is present in libc */
-#undef HAVE_SETRESGID		/* setresgid() is present in libc */
-#undef HAVE_SETEGID		/* setegid() is present in libc */
-#undef HAVE_SETGID		/* setgid() is present in libc */
-#undef HAVE_GETPGID		/* getpgid() is present in libc (POSIX) */
-#undef HAVE_SETPGID		/* setpgid() is present in libc (POSIX) */
-#undef HAVE_GETPGRP		/* getpgrp() is present in libc (ANY) */
-#undef HAVE_SETPGRP		/* setpgrp() is present in libc (ANY) */
-#undef HAVE_BSD_GETPGRP		/* getpgrp() in libc is BSD-4.2 compliant */
-#undef HAVE_BSD_SETPGRP		/* setpgrp() in libc is BSD-4.2 compliant */
-#undef HAVE_GETSPNAM		/* getspnam() in libc (SVR4 compliant) */
-#undef HAVE_GETSPWNAM		/* getspwnam() in libsec.a (HP-UX) */
-#undef HAVE_SYNC		/* sync() is present in libc */
-#undef HAVE_FSYNC		/* fsync() is present in libc */
-#undef HAVE_WAIT3		/* working wait3() is present in libc */
-#undef HAVE_WAIT4		/* wait4() is present in libc */
-#undef HAVE_WAITID		/* waitid() is present in libc */
-#undef HAVE_WAITPID		/* waitpid() is present in libc */
-#undef HAVE_GETHOSTID		/* gethostid() is present in libc */
-#undef HAVE_GETHOSTNAME		/* gethostname() is present in libc */
-#undef HAVE_GETDOMAINNAME	/* getdomainname() is present in libc */
-#undef HAVE_GETPAGESIZE		/* getpagesize() is present in libc */
-#undef HAVE_GETRUSAGE		/* getrusage() is present in libc */
-#undef HAVE_POLL		/* poll() is present in libc */
-#undef HAVE_SELECT		/* select() is present in libc */
-#undef HAVE_LCHOWN		/* lchown() is present in libc */
-#undef HAVE_BRK			/* brk() is present in libc */
-#undef HAVE_SBRK		/* sbrk() is present in libc */
-#undef HAVE_VA_COPY		/* va_copy() is present in varargs.h/stdarg.h */
-#undef HAVE__VA_COPY		/* __va_copy() is present in varargs.h/stdarg.h */
-#undef HAVE_DTOA		/* BSD-4.4 __dtoa() is present in libc */
-#undef HAVE_DTOA_R		/* BSD-4.4 __dtoa() with result ptr (reentrant) */
-#undef HAVE_GETCWD		/* POSIX getcwd() is present in libc */
-#undef HAVE_SMMAP		/* may map anonymous memory to get shared mem */
-#undef HAVE_SHMAT		/* shmat() is present in libc */
-#undef HAVE_SEMGET		/* semget() is present in libc */
-#undef HAVE_LSTAT		/* lstat() is present in libc */
-#undef HAVE_READLINK		/* readlink() is present in libc */
-#undef HAVE_LINK		/* link() is present in libc */
-#undef HAVE_RENAME		/* rename() is present in libc */
-#undef HAVE_MKFIFO		/* mkfifo() is present in libc */
-#undef HAVE_MKNOD		/* mknod() is present in libc */
-#undef HAVE_ECVT		/* ecvt() is present in libc */
-#undef HAVE_FCVT		/* fcvt() is present in libc */
-#undef HAVE_GCVT		/* gcvt() is present in libc */
-#undef HAVE_ECVT_R		/* ecvt_r() is present in libc */
-#undef HAVE_FCVT_R		/* fcvt_r() is present in libc */
-#undef HAVE_GCVT_R		/* gcvt_r() is present in libc */
-#undef HAVE_ECONVERT		/* econvert() is present in libc */
-#undef HAVE_FCONVERT		/* fconvert() is present in libc */
-#undef HAVE_GCONVERT		/* gconvert() is present in libc */
-#undef HAVE_ISINF		/* isinf() is present in libc */
-#undef HAVE_ISNAN		/* isnan() is present in libc */
-#undef HAVE_RAND		/* rand() is present in libc */
-#undef HAVE_DRAND48		/* drand48() is present in libc */
-#undef HAVE_SETPRIORITY		/* setpriority() is present in libc */
-#undef HAVE_NICE		/* nice() is present in libc */
-#undef HAVE_DOSSETPRIORITY	/* DosSetPriority() is present in libc */
-#undef HAVE_DOSALLOCSHAREDMEM	/* DosAllocSharedMem() is present in libc */
-#undef HAVE_SEEKDIR		/* seekdir() is present in libc */
-#undef HAVE_PUTENV		/* putenv() is present in libc (preferred function) */
-#undef HAVE_SETENV		/* setenv() is present in libc (use instead of putenv()) */
-#undef HAVE_UNAME		/* uname() is present in libc */
-#undef HAVE_SNPRINTF		/* snprintf() is present in libc */
-#undef HAVE_STRCASECMP		/* strcasecmp() is present in libc */
-#undef HAVE_STRSIGNAL		/* strsignal() is present in libc */
-#undef HAVE_STR2SIG		/* str2sig() is present in libc */
-#undef HAVE_SIG2STR		/* sig2str() is present in libc */
-#undef HAVE_KILLPG		/* killpg() is present in libc */
-#undef HAVE_SIGRELSE		/* sigrelse() is present in libc */
-#undef HAVE_SIGPROCMASK		/* sigprocmask() is present in libc (POSIX) */
-#undef HAVE_SIGSETMASK		/* sigsetmask() is present in libc (BSD) */
-#undef HAVE_SYS_SIGLIST		/* char *sys_siglist[] is present in libc */
-#undef HAVE_NANOSLEEP		/* nanosleep() is present in libc */
-#undef HAVE_USLEEP		/* usleep() is present in libc */
-#undef HAVE_FORK		/* fork() is present in libc */
-#undef HAVE_VALLOC		/* valloc() is present in libc (else use malloc())*/
+#undef HAVE_STRERROR
+#undef HAVE_MEMMOVE
+#undef HAVE_MLOCKALL
+#undef HAVE_MMAP
+#undef _MMAP_WITH_SIZEP
+#undef HAVE_FLOCK
+#undef HAVE_LOCKF
+#undef HAVE_FCNTL_LOCKF
+#undef HAVE_FCHDIR
+#undef HAVE_STATVFS
+#undef HAVE_QUOTACTL
+#undef HAVE_QUOTAIOCTL
+#undef HAVE_SETREUID
+#undef HAVE_SETRESUID
+#undef HAVE_SETEUID
+#undef HAVE_SETUID
+#undef HAVE_SETREGID
+#undef HAVE_SETRESGID
+#undef HAVE_SETEGID
+#undef HAVE_SETGID
+#undef HAVE_GETPGID
+#undef HAVE_SETPGID
+#undef HAVE_GETPGRP
+#undef HAVE_SETPGRP
+#undef HAVE_BSD_GETPGRP
+#undef HAVE_BSD_SETPGRP
+#undef HAVE_GETSPNAM
+#undef HAVE_GETSPWNAM
+#undef HAVE_SYNC
+#undef HAVE_FSYNC
+#undef HAVE_WAIT3
+#undef HAVE_WAIT4
+#undef HAVE_WAITID
+#undef HAVE_WAITPID
+#undef HAVE_GETHOSTID
+#undef HAVE_GETHOSTNAME
+#undef HAVE_GETDOMAINNAME
+#undef HAVE_GETPAGESIZE
+#undef HAVE_GETRUSAGE
+#undef HAVE_POLL
+#undef HAVE_SELECT
+#undef HAVE_LCHOWN
+#undef HAVE_BRK
+#undef HAVE_SBRK
+#undef HAVE_VA_COPY
+#undef HAVE__VA_COPY
+#undef HAVE_DTOA
+#undef HAVE_DTOA_R
+#undef HAVE_GETCWD
+#undef HAVE_SMMAP
+#undef HAVE_SHMAT
+#undef HAVE_SEMGET
+#undef HAVE_LSTAT
+#undef HAVE_READLINK
+#undef HAVE_LINK
+#undef HAVE_RENAME
+#undef HAVE_MKFIFO
+#undef HAVE_MKNOD
+#undef HAVE_ECVT
+#undef HAVE_FCVT
+#undef HAVE_GCVT
+#undef HAVE_ECVT_R
+#undef HAVE_FCVT_R
+#undef HAVE_GCVT_R
+#undef HAVE_ECONVERT
+#undef HAVE_FCONVERT
+#undef HAVE_GCONVERT
+#undef HAVE_ISINF
+#undef HAVE_ISNAN
+#undef HAVE_RAND
+#undef HAVE_DRAND48
+#undef HAVE_SETPRIORITY
+#undef HAVE_NICE
+#undef HAVE_DOSSETPRIORITY
+#undef HAVE_DOSALLOCSHAREDMEM
+#undef HAVE_SEEKDIR
+#undef HAVE_PUTENV
+#undef HAVE_SETENV
+#undef HAVE_UNAME
+#undef HAVE_SNPRINTF
+#undef HAVE_STRCASECMP
+#undef HAVE_STRSIGNAL
+#undef HAVE_STR2SIG
+#undef HAVE_SIG2STR
+#undef HAVE_KILLPG
+#undef HAVE_SIGRELSE
+#undef HAVE_SIGPROCMASK
+#undef HAVE_SIGSETMASK
+#undef HAVE_SYS_SIGLIST
+#undef HAVE_NANOSLEEP
+#undef HAVE_USLEEP
+#undef HAVE_FORK
+#undef HAVE_VALLOC
 #undef vfork
 
 #if	defined(HAVE_QUOTACTL) || defined(HAVE_QUOTAIOCTL)
@@ -227,35 +227,35 @@
 /*
  * Structures
  */
-#undef HAVE_MTGET_TYPE		/* if struct mtget contains mt_type (drive type) */
-#undef HAVE_MTGET_DSREG		/* if struct mtget contains mt_dsreg (drive status) */
-#undef HAVE_MTGET_ERREG		/* if struct mtget contains mt_erreg (error register) */
-#undef HAVE_MTGET_RESID		/* if struct mtget contains mt_resid (residual count) */
-#undef HAVE_MTGET_FILENO	/* if struct mtget contains mt_fileno (file #) */
-#undef HAVE_MTGET_BLKNO		/* if struct mtget contains mt_blkno (block #) */
-#undef HAVE_MTGET_FLAGS		/* if struct mtget contains mt_flags (flags) */
-#undef HAVE_MTGET_BF		/* if struct mtget contains mt_bf (optimum blocking factor) */
-#undef HAVE_STRUCT_RUSAGE	/* have struct rusage in sys/resource.h */
-#undef HAVE_UNION_SEMUN		/* have an illegal definition for union semun in sys/sem.h */
-#undef HAVE_UNION_WAIT		/* have union wait in wait.h */
-#undef HAVE_ST_SPARE1		/* if struct stat contains st_spare1 (usecs) */
-#undef HAVE_ST_NSEC		/* if struct stat contains st_atim.st_nsec (nanosecs */
-#undef HAVE_ST_BLKSIZE		/* if struct stat contains st_blksize */
-#undef HAVE_ST_BLOCKS		/* if struct stat contains st_blocks */
-#undef HAVE_ST_RDEV		/* if struct stat contains st_rdev */
-#undef STAT_MACROS_BROKEN	/* if the macros S_ISDIR, S_ISREG .. don't work */
+#undef HAVE_MTGET_TYPE
+#undef HAVE_MTGET_DSREG
+#undef HAVE_MTGET_ERREG
+#undef HAVE_MTGET_RESID
+#undef HAVE_MTGET_FILENO
+#undef HAVE_MTGET_BLKNO
+#undef HAVE_MTGET_FLAGS
+#undef HAVE_MTGET_BF
+#undef HAVE_STRUCT_RUSAGE
+#undef HAVE_UNION_SEMUN
+#undef HAVE_UNION_WAIT
+#undef HAVE_ST_SPARE1
+#undef HAVE_ST_NSEC
+#undef HAVE_ST_BLKSIZE
+#undef HAVE_ST_BLOCKS
+#undef HAVE_ST_RDEV
+#undef STAT_MACROS_BROKEN
 
-#undef DEV_MINOR_BITS		/* # if bits needed to hold minor device number */
-#undef DEV_MINOR_NONCONTIG	/* if bits in minor device number are noncontiguous */
+#undef DEV_MINOR_BITS
+#undef DEV_MINOR_NONCONTIG
 
 
 /*
  * Byteorder/Bitorder
  */
-#define	HAVE_C_BIGENDIAN	/* Flag that WORDS_BIGENDIAN test was done */
-#undef WORDS_BIGENDIAN		/* If using network byte order             */
-#define	HAVE_C_BITFIELDS	/* Flag that BITFIELDS_HTOL test was done  */
-#undef BITFIELDS_HTOL		/* If high bits come first in structures   */
+#define	HAVE_C_BIGENDIAN
+#undef WORDS_BIGENDIAN
+#define	HAVE_C_BITFIELDS
+#undef BITFIELDS_HTOL
 
 /*
  * Types/Keywords
@@ -273,20 +273,20 @@
 #undef SIZEOF_UNSIGNED_LONG_LONG
 #undef SIZEOF_UNSIGNED_CHAR_P
 
-#undef HAVE_LONGLONG		/* Compiler defines long long type */
-#undef CHAR_IS_UNSIGNED		/* Compiler defines char to be unsigned */
+#undef HAVE_LONGLONG
+#undef CHAR_IS_UNSIGNED
 
-#undef const			/* Define to empty if const doesn't work */
-#undef uid_t			/* To be used if uid_t is not present  */
-#undef size_t			/* To be used if size_t is not present */
-#undef pid_t			/* To be used if pid_t is not present  */
-#undef off_t			/* To be used if off_t is not present  */
-#undef mode_t			/* To be used if mode_t is not present */
-#undef time_t			/* To be used if time_t is not present */
-#undef caddr_t			/* To be used if caddr_t is not present */
-#undef daddr_t			/* To be used if daddr_t is not present */
-#undef dev_t			/* To be used if dev_t is not present */
-#undef socklen_t		/* To be used if socklen_t is not present */
+#undef const
+#undef uid_t
+#undef size_t
+#undef pid_t
+#undef off_t
+#undef mode_t
+#undef time_t
+#undef caddr_t
+#undef daddr_t
+#undef dev_t
+#undef socklen_t
 
 /*
  * These types are present on all UNIX systems but should be avoided
@@ -295,14 +295,14 @@
  *
  * Better include <utypes.h> and use Uchar, Uint & Ulong
  */
-#undef u_char			/* To be used if u_char is not present	*/
-#undef u_short			/* To be used if u_short is not present	*/
-#undef u_int			/* To be used if u_int is not present	*/
-#undef u_long			/* To be used if u_long is not present	*/
+#undef u_char
+#undef u_short
+#undef u_int
+#undef u_long
 
 /*#undef HAVE_SIZE_T*/
 /*#undef NO_SIZE_T*/
-#undef VA_LIST_IS_ARRAY		/* va_list is an array */
+#undef VA_LIST_IS_ARRAY
 #undef GETGROUPS_T
 #define GID_T		GETGROUPS_T
 
@@ -318,18 +318,18 @@
 
 #undef	HAVE_LARGEFILES
 
-#ifdef	HAVE_LARGEFILES		/* If we have working largefiles at all	   */
+#ifdef	HAVE_LARGEFILES
 				/* This is not defined with glibc-2.1.3	   */
 
-#undef _FILE_OFFSET_BITS	/* # of bits in off_t if settable	   */
-#undef _LARGEFILE_SOURCE	/* To make ftello() visible (HP-UX 10.20). */
-#undef _LARGE_FILES		/* Large file defined on AIX-style hosts.  */
-#undef _XOPEN_SOURCE		/* To make ftello() visible (glibc 2.1.3). */
+#undef _FILE_OFFSET_BITS
+#undef _LARGEFILE_SOURCE
+#undef _LARGE_FILES
+#undef _XOPEN_SOURCE
 				/* XXX We don't use this because glibc2.1.3*/
 				/* XXX is bad anyway. If we define	   */
 				/* XXX _XOPEN_SOURCE we will loose caddr_t */
 
-#undef HAVE_FSEEKO		/* Do we need this? If HAVE_LARGEFILES is  */
+#undef HAVE_FSEEKO
 				/* defined, we have fseeko()		   */
 
 #endif	/* HAVE_LARGEFILES */
@@ -338,13 +338,13 @@
 /*
  * Misc CC / LD related stuff
  */
-#undef NO_USER_MALLOC		/* If we cannot define our own malloc()	*/
+#undef NO_USER_MALLOC
 
 /*
  * Strings that help to maintain OS/platform id's in C-programs
  */
-#undef HOST_ALIAS		/* Output from config.guess (orig)	*/
-#undef HOST_SUB			/* Output from config.sub (modified)	*/
-#undef HOST_CPU			/* CPU part from HOST_SUB		*/
-#undef HOST_VENDOR		/* VENDOR part from HOST_SUB		*/
-#undef HOST_OS			/* CPU part from HOST_SUB		*/
+#undef HOST_ALIAS
+#undef HOST_SUB
+#undef HOST_CPU
+#undef HOST_VENDOR
+#undef HOST_OS
