diff -urN cdrtools-2.01.01.orig/autoconf/autoconf cdrtools-2.01.01/autoconf/autoconf
--- cdrtools-2.01.01.orig/autoconf/autoconf	2009-04-19 14:45:00.000000000 +0200
+++ cdrtools-2.01.01/autoconf/autoconf	2010-01-06 09:12:08.000000000 +0100
@@ -39,7 +39,7 @@
 
 : ${AC_MACRODIR=.}
 
-: ${M4=/usr/bin/gm4}
+: ${M4=%{_bindir}/m4}
 $M4 < /dev/null 2> /dev/null || M4=/opt/sfw/bin/gm4
 $M4 < /dev/null 2> /dev/null || M4=/usr/sfw/bin/gm4
 $M4 < /dev/null 2> /dev/null || M4=/opt/csw/bin/gm4
diff -urN cdrtools-2.01.01.orig/libsiconv/sic_nls.c cdrtools-2.01.01/libsiconv/sic_nls.c
--- cdrtools-2.01.01.orig/libsiconv/sic_nls.c	2009-11-08 20:58:13.000000000 +0100
+++ cdrtools-2.01.01/libsiconv/sic_nls.c	2010-01-06 09:11:57.000000000 +0100
@@ -176,7 +176,7 @@
 	struct dirent	*dp;
 	int		i = 0;
 
-	snprintf(path, sizeof (path), "%s/lib/siconv/", INS_BASE);
+	snprintf(path, sizeof (path), "%s/%{_lib}/siconv/", INS_BASE);
 	if ((d = opendir(path)) == NULL)
 		return (-1);
 
@@ -227,7 +227,7 @@
 	if (strchr(name, '/'))
 		return ((FILE *)NULL);
 
-	snprintf(path, sizeof (path), "%s/lib/siconv/%s", INS_BASE, name);
+	snprintf(path, sizeof (path), "%s/%{_lib}/siconv/%s", INS_BASE, name);
 	return (fopen(path, "r"));
 }
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10000.mk cdrtools-2.01.01/libsiconv/tables/cp10000.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10000.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10000.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10000
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10006.mk cdrtools-2.01.01/libsiconv/tables/cp10006.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10006.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10006.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10006
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10007.mk cdrtools-2.01.01/libsiconv/tables/cp10007.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10007.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10007.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10007
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10029.mk cdrtools-2.01.01/libsiconv/tables/cp10029.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10029.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10029.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10029
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10079.mk cdrtools-2.01.01/libsiconv/tables/cp10079.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10079.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10079.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10079
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp10081.mk cdrtools-2.01.01/libsiconv/tables/cp10081.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp10081.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp10081.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp10081
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1250.mk cdrtools-2.01.01/libsiconv/tables/cp1250.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1250.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1250.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1250
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1251.mk cdrtools-2.01.01/libsiconv/tables/cp1251.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1251.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1251.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1251
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1252.mk cdrtools-2.01.01/libsiconv/tables/cp1252.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1252.mk	2007-05-19 12:58:47.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1252.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1252
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1253.mk cdrtools-2.01.01/libsiconv/tables/cp1253.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1253.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1253.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1253
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1254.mk cdrtools-2.01.01/libsiconv/tables/cp1254.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1254.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1254.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1254
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1255.mk cdrtools-2.01.01/libsiconv/tables/cp1255.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1255.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1255.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1255
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1256.mk cdrtools-2.01.01/libsiconv/tables/cp1256.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1256.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1256.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1256
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1257.mk cdrtools-2.01.01/libsiconv/tables/cp1257.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1257.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1257.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1257
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp1258.mk cdrtools-2.01.01/libsiconv/tables/cp1258.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp1258.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp1258.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp1258
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp437.mk cdrtools-2.01.01/libsiconv/tables/cp437.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp437.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp437.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp437
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp737.mk cdrtools-2.01.01/libsiconv/tables/cp737.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp737.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp737.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp737
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp775.mk cdrtools-2.01.01/libsiconv/tables/cp775.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp775.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp775.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp775
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp850.mk cdrtools-2.01.01/libsiconv/tables/cp850.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp850.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp850.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp850
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp852.mk cdrtools-2.01.01/libsiconv/tables/cp852.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp852.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp852.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp852
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp855.mk cdrtools-2.01.01/libsiconv/tables/cp855.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp855.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp855.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp855
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp857.mk cdrtools-2.01.01/libsiconv/tables/cp857.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp857.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp857.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp857
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp860.mk cdrtools-2.01.01/libsiconv/tables/cp860.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp860.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp860.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp860
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp861.mk cdrtools-2.01.01/libsiconv/tables/cp861.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp861.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp861.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp861
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp862.mk cdrtools-2.01.01/libsiconv/tables/cp862.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp862.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp862.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp862
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp863.mk cdrtools-2.01.01/libsiconv/tables/cp863.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp863.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp863.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp863
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp864.mk cdrtools-2.01.01/libsiconv/tables/cp864.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp864.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp864.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp864
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp865.mk cdrtools-2.01.01/libsiconv/tables/cp865.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp865.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp865.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp865
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp866.mk cdrtools-2.01.01/libsiconv/tables/cp866.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp866.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp866.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp866
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp869.mk cdrtools-2.01.01/libsiconv/tables/cp869.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp869.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp869.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp869
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/cp874.mk cdrtools-2.01.01/libsiconv/tables/cp874.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/cp874.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/cp874.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		cp874
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-10.mk cdrtools-2.01.01/libsiconv/tables/iso8859-10.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-10.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-10.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-10
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-11.mk cdrtools-2.01.01/libsiconv/tables/iso8859-11.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-11.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-11.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-11
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-13.mk cdrtools-2.01.01/libsiconv/tables/iso8859-13.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-13.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-13.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-13
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-14.mk cdrtools-2.01.01/libsiconv/tables/iso8859-14.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-14.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-14.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-14
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-15.mk cdrtools-2.01.01/libsiconv/tables/iso8859-15.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-15.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-15.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-15
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-16.mk cdrtools-2.01.01/libsiconv/tables/iso8859-16.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-16.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-16.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-16
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-1.mk cdrtools-2.01.01/libsiconv/tables/iso8859-1.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-1.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-1.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-1
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-2.mk cdrtools-2.01.01/libsiconv/tables/iso8859-2.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-2.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-2.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-2
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-3.mk cdrtools-2.01.01/libsiconv/tables/iso8859-3.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-3.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-3.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-3
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-4.mk cdrtools-2.01.01/libsiconv/tables/iso8859-4.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-4.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-4.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-4
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-5.mk cdrtools-2.01.01/libsiconv/tables/iso8859-5.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-5.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-5.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-5
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-6.mk cdrtools-2.01.01/libsiconv/tables/iso8859-6.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-6.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-6.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-6
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-7.mk cdrtools-2.01.01/libsiconv/tables/iso8859-7.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-7.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-7.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-7
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-8.mk cdrtools-2.01.01/libsiconv/tables/iso8859-8.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-8.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-8.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-8
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/iso8859-9.mk cdrtools-2.01.01/libsiconv/tables/iso8859-9.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/iso8859-9.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/iso8859-9.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		iso8859-9
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/koi8-r.mk cdrtools-2.01.01/libsiconv/tables/koi8-r.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/koi8-r.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/koi8-r.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		koi8-r
 #XMK_FILE=	Makefile.man
 
diff -urN cdrtools-2.01.01.orig/libsiconv/tables/koi8-u.mk cdrtools-2.01.01/libsiconv/tables/koi8-u.mk
--- cdrtools-2.01.01.orig/libsiconv/tables/koi8-u.mk	2007-05-19 12:58:48.000000000 +0200
+++ cdrtools-2.01.01/libsiconv/tables/koi8-u.mk	2010-01-06 09:11:57.000000000 +0100
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		lib/siconv
+INSDIR=		%{_lib}/siconv
 TARGET=		koi8-u
 #XMK_FILE=	Makefile.man
 
