--- cdrtools-3.01/RULES/rules.cmd.orig	2020-12-28 20:54:28.689106159 +0100
+++ cdrtools-3.01/RULES/rules.cmd	2020-12-28 20:57:24.211488606 +0100
@@ -42,7 +42,7 @@
 all:		$(PTARGET)
 
 $(PTARGET):	$(OFILES) $(SRCLIBS)
-		$(LDCC) $(_LD_OUTPUT_OPTION) $(POFILES) $(LDFLAGS) $(LDLIBS)
+		$(LDCC) $(LDFLAGS) $(_LD_OUTPUT_OPTION) $(POFILES) $(LDLIBS)
 
 ###########################################################################
 include		$(SRCROOT)/$(RULESDIR)/sub.htm
--- cdrtools-3.01/RULES/rules.inc.orig	2020-12-28 20:54:28.689106159 +0100
+++ cdrtools-3.01/RULES/rules.inc	2020-12-28 20:57:57.307975973 +0100
@@ -41,7 +41,7 @@
 all:		$(PTARGETCX) $(PTARGET)
 
 $(PTARGETCX):	$(OFILES) $(SRCLIBS)
-		$(LDCC) $(_LD_OUTPUT_OPTION) $(POFILES) $(LDFLAGS) $(LDLIBS)
+		$(LDCC) $(LDFLAGS) $(_LD_OUTPUT_OPTION) $(POFILES) $(LDLIBS)
 #
 # If calling "$(PTARGETC)$(_XEXEEXT)" fails, this may be a cross compilation,
 # so retry to run the command under control of "$(CONFIG_RMTCALL)".
