diff -ruN cdrtools-2.0-/cdrecord/README.ATAPI cdrtools-2.0/cdrecord/README.ATAPI
--- cdrtools-2.0-/cdrecord/README.ATAPI	Tue Dec 24 17:37:35 2002
+++ cdrtools-2.0/cdrecord/README.ATAPI	Thu Dec 26 18:25:53 2002
@@ -462,14 +462,14 @@
       ln -s scd1 cdrw  # NOTE: see above, maybe cdrom
    Check the new SCSI settings:
       cdrecord -scanbus
-   Setup cdrecord's environment - edit /etc/default/cdrecord:
+   Setup cdrecord's environment - edit /etc/cdrecord.conf:
       CDR_DEVICE=cdrw
       cdrw=1,0,0        4       8m
       cdrom=1,2,0       0       0m
    Input the right values, the fields are described in the manpage
    of cdrecord. Alternatively, you may use this values as
    cdrecord-parameter or take a frontend with an own configuration
-   scheme, then you don't need to modify /etc/default/cdrecord.
+   scheme, then you don't need to modify /etc/cdrecord.conf
 9. It's done! Insert a CD and try "cdrecord -v -toc"
 /*--------------------------------------------------------------------------*/
 He had constant buffer underrun problems:
diff -ruN cdrtools-2.0-/README.openserver cdrtools-2.0/README.openserver
--- cdrtools-2.0-/README.openserver	Tue Jan 25 17:37:48 2000
+++ cdrtools-2.0/README.openserver	Thu Dec 26 18:25:53 2002
@@ -55,7 +55,7 @@
 ===================================================================
            THE OLD ADDRESSING IS NO LONGER AVAILABLE!
 	    
-         PLEASE ADJUST YOUR /etc/default/cdrecord FILES!
+         PLEASE ADJUST YOUR /etc/cdrecord.conf FILES!
 ===================================================================
 
 Running cdrecord with the old address method will result in an error!
diff -ruN cdrtools-2.0-/README.unixware cdrtools-2.0/README.unixware
--- cdrtools-2.0-/README.unixware	Sun Mar  4 00:41:43 2001
+++ cdrtools-2.0/README.unixware	Thu Dec 26 18:25:53 2002
@@ -64,7 +64,7 @@
 ===================================================================
            THE OLD ADDRESSING IS NO LONGER AVAILABLE!
 	    
-         PLEASE ADJUST YOUR /etc/default/cdrecord FILES!
+         PLEASE ADJUST YOUR /etc/cdrecord.conf FILES!
 ===================================================================
 
 Running cdrecord with the old address method will result in an error!
diff -ruN cdrtools-2.0-/cdrecord/cdrecord.1 cdrtools-2.0/cdrecord/cdrecord.1
--- cdrtools-2.0-/cdrecord/cdrecord.1	Tue Dec 24 16:28:40 2002
+++ cdrtools-2.0/cdrecord/cdrecord.1	Thu Dec 26 18:25:53 2002
@@ -146,7 +146,7 @@
 described below.
 
 .PP
-If a file /etc/default/cdrecord exists, the parameter to the
+If a file /etc/cdrecord.conf exists, the parameter to the
 .B dev=
 option may also be a drive name label in said file (see FILES section).
 
@@ -621,7 +621,7 @@
 .B dev=
 option does not contain the characters ',', '/', '@' or ':',
 it is interpreted as an label name that may be found in the file
-/etc/default/cdrecord (see FILES section).
+/etc/cdrecord.conf (see FILES section).
 .sp
 If no 
 .B dev=
@@ -1361,7 +1361,7 @@
 .TP
 .B CDR_DEVICE
 This may either hold a device identifier that is suitable to the open
-call of the SCSI transport library or a label in the file /etc/default/cdrecord.
+call of the SCSI transport library or a label in the file /etc/cdrecord.conf
 .TP
 .B CDR_SPEED
 Sets the default speed value for writing (see also 
@@ -1417,8 +1417,11 @@
 
 .SH FILES
 .TP
-/etc/default/cdrecord
-Default values can be set for the following options in /etc/default/cdrecord.
+/etc/cdrecord.conf
+Default values can be set for the following options in /etc/cdrecord.conf .
+Note that this path has been changed from standard location in cdrtools
+distribution (/etc/default/cdrecord) to be consistent with other packages
+which don't use /etc/default directory in general.
 For example:
 .SM CDR_FIFOSIZE=8m
 or
@@ -1427,7 +1430,7 @@
 .TP
 CDR_DEVICE
 This may either hold a device identifier that is suitable to the open
-call of the SCSI transport library or a label in the file /etc/default/cdrecord 
+call of the SCSI transport library or a label in the file /etc/cdrecord.conf 
 that allows to identify a specific drive on the system.
 .TP
 CDR_SPEED
diff -ruN cdrtools-2.0-/cdrecord/cdrecord.dfl cdrtools-2.0/cdrecord/cdrecord.dfl
--- cdrtools-2.0-/cdrecord/cdrecord.dfl	Sun Jul  7 23:56:21 2002
+++ cdrtools-2.0/cdrecord/cdrecord.dfl	Thu Dec 26 18:25:53 2002
@@ -1,6 +1,6 @@
 #ident @(#)cdrecord.dfl	1.4 02/07/07 Copyr 1998 J. Schilling
 #
-# This file is /etc/default/cdrecord
+# This file is /etc/cdrecord.conf
 # It contains defaults that are used if no command line option
 # or environment is present.
 # 
@@ -14,7 +14,7 @@
 # Note that newer cdrecord versions do not default
 # to speed=1. For MMC compliant drives, the default
 # is to write at maximum speed, so it in general does
-# not make sense to set up a default speed in /etc/default/cdrecord 
+# not make sense to set up a default speed in /etc/cdrecord.conf 
 #
 #CDR_SPEED=40
 
diff -ruN cdrtools-2.0-/libcdrdeflt/cdrdeflt.c cdrtools-2.0/libcdrdeflt/cdrdeflt.c
--- cdrtools-2.0-/libcdrdeflt/cdrdeflt.c	Wed Aug 14 19:52:00 2002
+++ cdrtools-2.0/libcdrdeflt/cdrdeflt.c	Thu Dec 26 18:25:53 2002
@@ -53,7 +53,7 @@
 	 * don't contribute work and don't give support, they are causing extra
 	 * work for me and this way slow down the cdrecord development.
 	 */
-	return (defltopen("/etc/default/cdrecord"));
+	return (defltopen("/etc/cdrecord.conf"));
 }
 
 EXPORT void
diff -ruN cdrtools-2.0-/doc/cdrecord.man cdrtools-2.0/doc/cdrecord.man
--- cdrtools-2.0-/doc/cdrecord.man	Tue Dec 24 16:54:52 2002
+++ cdrtools-2.0/doc/cdrecord.man	Thu Dec 26 18:28:09 2002
@@ -67,7 +67,7 @@
        try the --ssccaannbbuuss option of ccddrreeccoorrdd described below.
 
 
-       If  a  file /etc/default/cdrecord exists, the parameter to
+       If  a  file /etc/cdrecord.conf exists, the parameter to
        the ddeevv== option may also be a drive  name  label  in  said
        file (see FILES section).
 
@@ -438,7 +438,7 @@
               If the argument to the ddeevv== option does not contain
               the  characters  ',', '/', '@' or ':', it is inter-
               preted as an label name that may be  found  in  the
-              file /etc/default/cdrecord (see FILES section).
+              file /etc/cdrecord.conf (see FILES section).
 
        ggrraacceettiimmee==_#
               Set  the  grace  time before starting to write to _#
@@ -1057,7 +1057,7 @@
               This  may  either  hold a device identifier that is
               suitable to the open call  of  the  SCSI  transport
               library     or     a     label    in    the    file
-              /etc/default/cdrecord.
+              /etc/cdrecord.conf
 
        CCDDRR__SSPPEEEEDD
               Sets the default speed value for writing (see  also
@@ -1096,16 +1096,16 @@
 
 
 FFIILLEESS
-       /etc/default/cdrecord
+       /etc/cdrecord.conf
               Default values can be set for the following options
-              in  /etc/default/cdrecord.  For example: CDR_FIFOS-
+              in   /etc/cdrecord.conf   For  example:  CDR_FIFOS-
               IZE=8m or CDR_SPEED=2
 
               CDR_DEVICE
                      This may either  hold  a  device  identifier
                      that  is  suitable  to  the open call of the
                      SCSI transport library or  a  label  in  the
-                     file  /etc/default/cdrecord  that  allows to
+                     file   /etc/cdrecord.conf   that  allows  to
                      identify a specific drive on the system.
 
               CDR_SPEED
diff -ruN cdrtools-2.0-/doc/cdrecord.ps cdrtools-2.0/doc/cdrecord.ps
--- cdrtools-2.0-/doc/cdrecord.ps	Tue Dec 24 16:55:27 2002
+++ cdrtools-2.0/doc/cdrecord.ps	Thu Dec 26 18:26:59 2002
@@ -290,7 +290,7 @@
 (,).68 E F3(lun).09 E F0 2.158(should be used, try the)4.658 F F2
 (\255scanb)4.658 E(us)-.2 E F0 2.158(option of)4.658 F F2(cdr)108 525.6
 Q(ecord)-.18 E F0(described belo)2.5 E -.65(w.)-.25 G .114
-(If a \214le /etc/def)108 554.4 R .114(ault/cdrecord e)-.1 F .115
+(If a \214le /etc/cdr)108 554.4 R .114(ecord.conf e)-.1 F .115
 (xists, the parameter to the)-.15 F F2(de)2.615 E(v=)-.15 E F0 .115
 (option may also be a dri)2.615 F .415 -.15(ve n)-.25 H .115
 (ame label in said).15 F(\214le \(see FILES section\).)108 566.4 Q(On)
@@ -749,8 +749,8 @@
 556.8 R 1.047(gument to the)-.18 F F2(de)3.547 E(v=)-.15 E F0 1.047
 (option does not contain the characters ',)3.547 F 1.047
 (', '/', '@' or ':', it is inter)-.7 F(-)-.2 E
-(preted as an label name that may be found in the \214le /etc/def)144
-568.8 Q(ault/cdrecord \(see FILES section\).)-.1 E F2(gracetime=)108
+(preted as an label name that may be found in the \214le /etc/cdr)144
+568.8 Q(ecord.conf \(see FILES section\).)-.1 E F2(gracetime=)108
 585.6 Q F1(#).48 E F0(Set the grace time before starting to write to)144
 597.6 Q F1(#)2.5 E F0 2.5(seconds. V)3.4 F(alues belo)-1.11 E 2.5(w2s)
 -.25 G(econds are not allo)-2.5 E(wed.)-.25 E F2(timeout=)108 614.4 Q F1
@@ -1430,8 +1430,8 @@
 G(nstead.)-2.5 E/F3 10.95/Times-Bold@0 SF(ENVIR)72 309.6 Q(ONMENT)-.329
 E F2(CDR_DEVICE)108 321.6 Q F0 1.628(This may either hold a de)144 333.6
 R 1.628(vice identi\214er that is suitable to the open call of the SCSI\
- transport)-.25 F(library or a label in the \214le /etc/def)144 345.6 Q
-(ault/cdrecord.)-.1 E F2(CDR_SPEED)108 362.4 Q F0(Sets the def)144 374.4
+ transport)-.25 F(library or a label in the \214le /etc/cdr)144 345.6 Q
+(ecord.conf)-.1 E F2(CDR_SPEED)108 362.4 Q F0(Sets the def)144 374.4
 Q(ault speed v)-.1 E(alue for writing \(see also)-.25 E F2(\255speed)2.5
 E F0(option\).)2.5 E F2(CDR_FIFOSIZE)108 391.2 Q F0(Sets the def)144
 403.2 Q(ault size of the FIFO \(see also)-.1 E F2(fs=)2.5 E F0 2.5(#o)C
@@ -1465,16 +1465,16 @@
 (ote that the remote SCSI serv)-6.236 F(er)-.15 E .553(program name wil\
 l be ignored if you log in using an account that has been created with \
 a remote)144 585.6 R(SCSI serv)144 597.6 Q(er program as login shell.)
--.15 E F3(FILES)72 626.4 Q F0(/etc/def)108 638.4 Q(ault/cdrecord)-.1 E
+-.15 E F3(FILES)72 626.4 Q F0(/etc/rec)108 638.4 Q(ecord.conf)-.1 E
 (Def)144 650.4 Q 3.228(ault v)-.1 F 3.228
-(alues can be set for the follo)-.25 F 3.227(wing options in /etc/def)
--.25 F 5.727(ault/cdrecord. F)-.1 F 3.227(or e)-.15 F(xample:)-.15 E/F4
+(alues can be set for the follo)-.25 F 3.227(wing options in /etc/rec)
+-.25 F 5.727(ecord.conf F)-.1 F 3.227(or e)-.15 F(xample:)-.15 E/F4
 9/Times-Roman@0 SF(CDR_FIFOSIZE=8m)144 662.4 Q F0(or)2.25 E F4
 (CDR_SPEED=2)2.5 E F0(CDR_DEVICE)144 679.2 Q .261
 (This may either hold a de)180 691.2 R .262
 (vice identi\214er that is suitable to the open call of the SCSI trans-)
--.25 F 1.418(port library or a label in the \214le /etc/def)180 703.2 R
-1.418(ault/cdrecord that allo)-.1 F 1.418(ws to identify a speci\214c)
+-.25 F 1.418(port library or a label in the \214le /etc/rec)180 703.2 R
+1.418(ecord.conf that allo)-.1 F 1.418(ws to identify a speci\214c)
 -.25 F(dri)180 715.2 Q .3 -.15(ve o)-.25 H 2.5(nt).15 G(he system.)-2.5
 E(J)72 768 Q -.25(..)-5.5 K(or)-4.5 5.5 M 2.5(gS)-.18 G 153.065
 (chilling V)-2.5 F(ersion 2.0)-1.11 E(12)201.225 E EP
diff -ruN cdrtools-2.0-/doc/readcd.man cdrtools-2.0/doc/readcd.man
--- cdrtools-2.0-/doc/readcd.man	Tue Dec 24 16:56:07 2002
+++ cdrtools-2.0/doc/readcd.man	Thu Dec 26 18:25:53 2002
@@ -96,7 +96,7 @@
               If the argument to the ddeevv== option does not contain
               the  characters  ',', '/', '@' or ':', it is inter-
               preted as an label name that may be  found  in  the
-              file /etc/default/cdrecord (see FILES section).
+              file /etc/cdrecord.conf (see FILES section).
 
        ttiimmeeoouutt==_#
               Set  the  default  SCSI  command timeout value to _#
diff -ruN cdrtools-2.0-/doc/readcd.ps cdrtools-2.0/doc/readcd.ps
--- cdrtools-2.0-/doc/readcd.ps	Tue Dec 24 16:56:28 2002
+++ cdrtools-2.0/doc/readcd.ps	Thu Dec 26 18:25:53 2002
@@ -330,8 +330,8 @@
 120 R 1.046(gument to the)-.18 F F2(de)3.547 E(v=)-.15 E F0 1.047
 (option does not contain the characters ',)3.547 F 1.047
 (', '/', '@' or ':', it is inter)-.7 F(-)-.2 E
-(preted as an label name that may be found in the \214le /etc/def)144
-132 Q(ault/cdrecord \(see FILES section\).)-.1 E F2(timeout=)108 148.8 Q
+(preted as an label name that may be found in the \214le /etc/cdr)144
+132 Q(ecord.conf \(see FILES section\).)-.1 E F2(timeout=)108 148.8 Q
 F1(#).48 E F0 .13(Set the def)144 160.8 R .13
 (ault SCSI command timeout v)-.1 F .13(alue to)-.25 F F1(#)2.63 E F0
 2.63(seconds. The)3.53 F(def)2.63 E .13(ault SCSI command timeout is)-.1
diff -ruN cdrtools-2.0-/doc/scgcheck.man cdrtools-2.0/doc/scgcheck.man
--- cdrtools-2.0-/doc/scgcheck.man	Tue Dec 24 16:56:55 2002
+++ cdrtools-2.0/doc/scgcheck.man	Thu Dec 26 18:25:53 2002
@@ -90,7 +90,7 @@
               If the argument to the ddeevv== option does not contain
               the characters ',', '/', '@' or ':', it  is  inter-
               preted  as  an  label name that may be found in the
-              file /etc/default/cdrecord (see FILES section).
+              file /etc/cdrecord.conf (see FILES section).
 
        ttiimmeeoouutt==_#
               Set the default SCSI command  timeout  value  to  _#
diff -ruN cdrtools-2.0-/doc/scgcheck.ps cdrtools-2.0/doc/scgcheck.ps
--- cdrtools-2.0-/doc/scgcheck.ps	Tue Dec 24 16:57:15 2002
+++ cdrtools-2.0/doc/scgcheck.ps	Thu Dec 26 18:25:53 2002
@@ -317,8 +317,8 @@
 1.047(gument to the)-.18 F/F1 10/Times-Bold@0 SF(de)3.547 E(v=)-.15 E F0
 1.047(option does not contain the characters ',)3.547 F 1.047
 (', '/', '@' or ':', it is inter)-.7 F(-)-.2 E
-(preted as an label name that may be found in the \214le /etc/def)144 96
-Q(ault/cdrecord \(see FILES section\).)-.1 E F1(timeout=)108 112.8 Q/F2
+(preted as an label name that may be found in the \214le /etc/cdr)144 96
+Q(ecord.conf \(see FILES section\).)-.1 E F1(timeout=)108 112.8 Q/F2
 10/Times-Italic@0 SF(#).48 E F0 .13(Set the def)144 124.8 R .13
 (ault SCSI command timeout v)-.1 F .13(alue to)-.25 F F2(#)2.63 E F0
 2.63(seconds. The)3.53 F(def)2.63 E .13(ault SCSI command timeout is)-.1
diff -ruN cdrtools-2.0-/readcd/readcd.1 cdrtools-2.0/readcd/readcd.1
--- cdrtools-2.0-/readcd/readcd.1	Tue Dec 24 16:28:41 2002
+++ cdrtools-2.0/readcd/readcd.1	Thu Dec 26 18:25:53 2002
@@ -186,7 +186,7 @@
 .B dev=
 option does not contain the characters ',', '/', '@' or ':',
 it is interpreted as an label name that may be found in the file
-/etc/default/cdrecord (see FILES section).
+/etc/cdrecord.conf (see FILES section).
 .sp
 If no 
 .B dev=
diff -ruN cdrtools-2.0-/scgcheck/scgcheck.1 cdrtools-2.0/scgcheck/scgcheck.1
--- cdrtools-2.0-/scgcheck/scgcheck.1	Tue Dec 24 16:32:01 2002
+++ cdrtools-2.0/scgcheck/scgcheck.1	Thu Dec 26 18:25:53 2002
@@ -170,7 +170,7 @@
 .B dev=
 option does not contain the characters ',', '/', '@' or ':',
 it is interpreted as an label name that may be found in the file
-/etc/default/cdrecord (see FILES section).
+/etc/cdrecord.conf (see FILES section).
 .TP
 .BI timeout= #
 Set the default SCSI command timeout value to 
