--- cdrtools-3.01/DEFAULTS/Defaults.linux.orig	2020-12-28 19:35:02.348260976 +0100
+++ cdrtools-3.01/DEFAULTS/Defaults.linux	2020-12-28 20:53:15.129504665 +0100
@@ -40,17 +40,12 @@
 # LINUX_SRC_INCLUDE is used in RULES/*-linux-*cc*.rul
 #
 ###########################################################################
-LINUX_INCL_PATH=	/usr/src/linux/include
-
-_LINUX_SRC_INCLUDE=	$(_UNIQ)$(MAY_LINUX_SRC_INCLUDE)
-__LINUX_SRC_INCLUDE=	$(_LINUX_SRC_INCLUDE:$(_UNIQ)=$(LINUX_INCL_PATH))
-LINUX_SRC_INCLUDE=      $(__LINUX_SRC_INCLUDE:$(_UNIQ)no%=%)
+LINUX_SRC_INCLUDE=
 
 DEFINCDIRS=	$(SRCROOT)/include
 DEFOSINCDIRS=
-LDPATH=		-L/opt/schily/lib
-#RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib -R$(OLIBSDIR)
-RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib
+LDPATH=
+RUNPATH=
 
 ###########################################################################
 #
--- cdrtools-3.01/RULES/rules.cnf.orig	2020-12-28 19:35:02.348260976 +0100
+++ cdrtools-3.01/RULES/rules.cnf	2020-12-28 20:53:58.149271607 +0100
@@ -64,7 +64,7 @@
 		cd $(OINCSDIR); \
 		CONFIG_NOFAIL=TRUE CC="$(XCC_COM)" CFLAGS="$(CFLAGS)" \
 		CXXFLAGS="$(C++FLAGS)" CPPFLAGS="$(OS_DEFS)" \
-		LDFLAGS="$(LDOPTS)" sh ../../autoconf/configure $(CONFFLAGS)
+		LDFLAGS="$(LDFLAGS1) $(LDOPTS)" sh ../../autoconf/configure $(CONFFLAGS)
 ###########################################################################
 include $(OINCSDIR)/Inull
 include $(OINCSDIR)/rules.cnf
