--- cdrtools-1.11/conf/xconfig.h.in.orig	Sat Mar 23 21:39:26 2002
+++ cdrtools-1.11/conf/xconfig.h.in	Wed Apr 24 21:32:02 2002
@@ -8,95 +8,95 @@
 /*
  * Header Files
  */
-#undef PROTOTYPES	/* if Compiler supports ANSI C prototypes */
-#undef HAVE_STDARG_H	/* to use stdarg.h, else use varargs.h NOTE: SaberC on a Sun has prototypes but no stdarg.h */
-#undef HAVE_VARARGS_H	/* to use use varargs.h NOTE: The free HP-UX C-compiler has stdarg.h but no PROTOTYPES */
-#undef HAVE_STDLIB_H	/* to use general utility defines (malloc(), size_t ...) and general C library prototypes */
-#undef HAVE_STRING_H	/* to get NULL and ANSI C string function prototypes */
-#undef HAVE_STRINGS_H	/* to get BSD string function prototypes */
-#undef STDC_HEADERS	/* if ANSI compliant stdlib.h, stdarg.h, string.h, float.h are present */
-#undef HAVE_UNISTD_H	/* to get POSIX syscall prototypes XXX sys/file.h fcntl.h (unixstd/fctl)XXX*/
-#undef HAVE_GETOPT_H	/* to get getopt() prototype from getopt.h instead of unistd.h */
-#undef HAVE_LIMITS_H	/* to get POSIX numeric limits constants */
-#undef HAVE_A_OUT_H	/* if a.out.h is present (may be a system using a.out format) */
-#undef HAVE_AOUTHDR_H	/* if aouthdr.h is present. This is a COFF system */
-#undef HAVE_ELF_H	/* if elf.h is present. This is an ELF system */
-#undef HAVE_FCNTL_H	/* to access, O_XXX constants for open(), otherwise use sys/file.h */
-#undef HAVE_SYS_FILE_H	/* to use O_XXX constants for open() and flock() defs */
-#undef HAVE_INTTYPES_H	/* to use UNIX-98 inttypes.h */
-#undef HAVE_DIRENT_H	/* to use POSIX dirent.h */
-#undef HAVE_SYS_DIR_H	/* to use BSD sys/dir.h */
-#undef HAVE_NDIR_H	/* to use ndir.h */
-#undef HAVE_SYS_NDIR_H	/* to use sys/ndir.h */
-#undef HAVE_MALLOC_H	/* if malloc.h exists */
-#undef HAVE_TERMIOS_H	/* to use POSIX termios.h */
-#undef HAVE_TERMIO_H	/* to use SVR4 termio.h */
-#undef HAVE_PWD_H	/* if pwd.h exists */
-#undef HAVE_GRP_H	/* if grp.h exists */
-#undef HAVE_SYS_ACL_H	/* to use <sys/acl.h> for ACL definitions */
-#undef HAVE_ACLLIB_H	/* if HP-UX <acllib.h> is present */
-#undef HAVE_SHADOW_H	/* if shadow.h exists */
-#undef HAVE_SYSLOG_H	/* if syslog.h exists */
-#undef HAVE_SYS_TIME_H	/* may include sys/time.h for struct timeval */
-#undef TIME_WITH_SYS_TIME	/* may include both time.h and sys/time.h */
-#undef HAVE_TIMES	/* to use times() and sys/times.h */
-#undef HAVE_SYS_TIMES_H	/* may include sys/times.h for struct tms */
-#undef HAVE_UTIMES		/* to use BSD utimes() and sys/time.h */
-#undef HAVE_UTIME_H		/* to use utime.h for the utimbuf structure declaration, else declare struct utimbuf yourself */
-#undef HAVE_SYS_UTIME_H		/* to use sys/utime.h if utime.h does not exist */
-#undef HAVE_SYS_IOCTL_H		/* if sys/ioctl.h is present */
-#undef HAVE_SYS_FILIO_H		/* if sys/ioctl.h is present */
-#undef HAVE_SYS_PARAM_H		/* if sys/param.h is present */
-#undef HAVE_MNTENT_H		/* if mntent.h is present */
-#undef HAVE_SYS_MNTENT_H	/* if sys/mntent.h is present */
-#undef HAVE_SYS_MNTTAB_H	/* if sys/mnttab.h is present */
-#undef HAVE_SYS_MOUNT_H		/* if sys/mount.h is present */
-#undef HAVE_WAIT_H		/* to use wait.h for prototypes and union wait */
-#undef HAVE_SYS_WAIT_H		/* else use sys/wait.h */
-#undef HAVE_SYS_RESOURCE_H	/* to use sys/resource.h for rlimit() and wait3() */
-#undef HAVE_SYS_PROCFS_H	/* to use sys/procfs.h for wait3() emulation */
-#undef HAVE_SYS_SYSTEMINFO_H	/* to use SVr4 sysinfo() */
-#undef HAVE_SYS_UTSNAME_H	/* to use uname() */
-#undef HAVE_SYS_PRIOCNTL_H	/* to use SVr4 priocntl() instead of nice()/setpriority() */
-#undef HAVE_SYS_RTPRIOCNTL_H	/* if the system supports SVr4 real time classes */
-#undef HAVE_SYS_SYSCALL_H	/* to use syscall() */
-#undef HAVE_SYS_MTIO_H		/* to use mtio definitions from sys/mtio.h */
-#undef HAVE_SYS_MMAN_H		/* to use definitions for mmap()/madvise()... from sys/mman.h */
-#undef HAVE_SYS_SHM_H		/* to use definitions for shmget() ... from sys/shm.h */
-#undef HAVE_SYS_IPC_H		/* to use definitions for shmget() ... from sys/ipc.h */
-#undef MAJOR_IN_MKDEV		/* if we should include sys/mkdev.h to get major()/minor()/makedev() */
-#undef MAJOR_IN_SYSMACROS	/* if we should include sys/sysmacros.h to get major()/minor()/makedev() */
-#undef HAVE_SYS_DKIO_H		/* if we may include sys/dkio.h for disk ioctls */
-#undef HAVE_SYS_DKLABEL_H	/* if we may include sys/dklabel.h for disk label */
-#undef HAVE_SUN_DKIO_H		/* if we may include sun/dkio.h for disk ioctls */
-#undef HAVE_SUN_DKLABEL_H	/* if we may include sun/dklabel.h for disk label */
-#undef HAVE_SYS_TYPES_H		/* if we may include sys/types.h (the standard) */
-#undef HAVE_SYS_STAT_H		/* if we may include sys/stat.h (the standard) */
-#undef HAVE_TYPES_H		/* if we may include types.h (rare cases e.g. ATARI TOS) */
-#undef HAVE_STAT_H		/* if we may include stat.h (rare cases e.g. ATARI TOS) */
-#undef HAVE_POLL_H		/* if we may include poll.h to use poll() */
-#undef HAVE_SYS_POLL_H		/* if we may include sys/poll.h to use poll() */
-#undef HAVE_SYS_SELECT_H	/* if we may have sys/select.h nonstandard use for select() on some systems*/
-#undef NEED_SYS_SELECT_H	/* if we need sys/select.h to use select() (this is nonstandard) */
-#undef HAVE_NETDB_H		/* if we have netdb.h for get*by*() and rcmd() */
-#undef HAVE_SYS_SOCKET_H	/* if we have sys/socket.h for socket() */
-#undef NEED_SYS_SOCKET_H	/* if we need sys/socket.h to use select() (this is nonstandard) */
-#undef HAVE_LINUX_PG_H		/* if we may include linux/pg.h for PP ATAPI sypport */
-#undef HAVE_CAMLIB_H		/* if we may include camlib.h for CAM SCSI transport definitions */
-#undef HAVE_IEEEFP_H		/* if we may include ieeefp.h for finite()/isnand() */
-#undef HAVE_FP_H		/* if we may include fp.h for FINITE()/IS_INF()/IS_NAN() */
-#undef HAVE_VALUES_H		/* if we may include values.h for MAXFLOAT */
-#undef HAVE_FLOAT_H		/* if we may include float.h for FLT_MAX */
-#undef HAVE__FILBUF		/* if we have _filbuf() for USG derived STDIO */
-#undef HAVE___FILBUF		/* if we have __filbuf() for USG derived STDIO */
-#undef HAVE_USG_STDIO		/* if we have USG derived STDIO */
-#undef HAVE_ERRNO_DEF		/* if we have errno definition in <errno.h> */
-#undef HAVE_VFORK_H		/* if we should include vfork.h for vfork() definitions */
-#undef HAVE_ARPA_INET_H		/* if we have arpa/inet.h (missing on BeOS) */
-				/* BeOS has inet_ntoa() in <netdb.h> */
-#undef HAVE_BSD_DEV_SCSIREG_H	/* if we have a NeXT Step compatible sg driver */
-#undef HAVE_SYS_BSDTTY_H	/* if we have sys/bsdtty.h on HP-UX for TIOCGPGRP */
-#undef HAVE_OS_H		/* if we have the BeOS kernel definitions in OS.h */
+#undef PROTOTYPES
+#undef HAVE_STDARG_H
+#undef HAVE_VARARGS_H
+#undef HAVE_STDLIB_H
+#undef HAVE_STRING_H
+#undef HAVE_STRINGS_H
+#undef STDC_HEADERS
+#undef HAVE_UNISTD_H
+#undef HAVE_GETOPT_H
+#undef HAVE_LIMITS_H
+#undef HAVE_A_OUT_H
+#undef HAVE_AOUTHDR_H
+#undef HAVE_ELF_H
+#undef HAVE_FCNTL_H
+#undef HAVE_SYS_FILE_H
+#undef HAVE_INTTYPES_H
+#undef HAVE_DIRENT_H
+#undef HAVE_SYS_DIR_H
+#undef HAVE_NDIR_H
+#undef HAVE_SYS_NDIR_H
+#undef HAVE_MALLOC_H
+#undef HAVE_TERMIOS_H
+#undef HAVE_TERMIO_H
+#undef HAVE_PWD_H
+#undef HAVE_GRP_H
+#undef HAVE_SYS_ACL_H
+#undef HAVE_ACLLIB_H
+#undef HAVE_SHADOW_H
+#undef HAVE_SYSLOG_H
+#undef HAVE_SYS_TIME_H
+#undef TIME_WITH_SYS_TIME
+#undef HAVE_TIMES
+#undef HAVE_SYS_TIMES_H
+#undef HAVE_UTIMES
+#undef HAVE_UTIME_H
+#undef HAVE_SYS_UTIME_H
+#undef HAVE_SYS_IOCTL_H
+#undef HAVE_SYS_FILIO_H
+#undef HAVE_SYS_PARAM_H
+#undef HAVE_MNTENT_H
+#undef HAVE_SYS_MNTENT_H
+#undef HAVE_SYS_MNTTAB_H
+#undef HAVE_SYS_MOUNT_H
+#undef HAVE_WAIT_H
+#undef HAVE_SYS_WAIT_H
+#undef HAVE_SYS_RESOURCE_H
+#undef HAVE_SYS_PROCFS_H
+#undef HAVE_SYS_SYSTEMINFO_H
+#undef HAVE_SYS_UTSNAME_H
+#undef HAVE_SYS_PRIOCNTL_H
+#undef HAVE_SYS_RTPRIOCNTL_H
+#undef HAVE_SYS_SYSCALL_H
+#undef HAVE_SYS_MTIO_H
+#undef HAVE_SYS_MMAN_H
+#undef HAVE_SYS_SHM_H
+#undef HAVE_SYS_IPC_H
+#undef MAJOR_IN_MKDEV
+#undef MAJOR_IN_SYSMACROS
+#undef HAVE_SYS_DKIO_H
+#undef HAVE_SYS_DKLABEL_H
+#undef HAVE_SUN_DKIO_H
+#undef HAVE_SUN_DKLABEL_H
+#undef HAVE_SYS_TYPES_H
+#undef HAVE_SYS_STAT_H
+#undef HAVE_TYPES_H
+#undef HAVE_STAT_H
+#undef HAVE_POLL_H
+#undef HAVE_SYS_POLL_H
+#undef HAVE_SYS_SELECT_H
+#undef NEED_SYS_SELECT_H
+#undef HAVE_NETDB_H
+#undef HAVE_SYS_SOCKET_H
+#undef NEED_SYS_SOCKET_H
+#undef HAVE_LINUX_PG_H
+#undef HAVE_CAMLIB_H
+#undef HAVE_IEEEFP_H
+#undef HAVE_FP_H
+#undef HAVE_VALUES_H
+#undef HAVE_FLOAT_H
+#undef HAVE__FILBUF
+#undef HAVE___FILBUF
+#undef HAVE_USG_STDIO
+#undef HAVE_ERRNO_DEF
+#undef HAVE_VFORK_H
+#undef HAVE_ARPA_INET_H
+
+#undef HAVE_BSD_DEV_SCSIREG_H
+#undef HAVE_SYS_BSDTTY_H
+#undef HAVE_OS_H
 
 /*
  * Convert to SCHILY name
@@ -108,13 +108,13 @@
 #endif
 
 #ifdef	HAVE_ELF_H
-#define	HAVE_ELF			/* This system uses ELF */
+#define	HAVE_ELF
 #else
 #	ifdef	HAVE_AOUTHDR_H
-#	define	HAVE_COFF		/* This system uses COFF */
+#	define	HAVE_COFF
 #	else
 #		ifdef HAVE_A_OUT_H
-#		define HAVE_AOUT	/* This system uses AOUT */
+#		define HAVE_AOUT
 #		endif
 #	endif
 #endif
@@ -122,122 +122,122 @@
 /*
  * Library Functions
  */
-#undef HAVE_STRERROR		/* strerror() is present in libc */
-#undef HAVE_MEMMOVE		/* memmove() is present in libc */
-#undef HAVE_MADVISE		/* madvise() is present in libc */
-#undef HAVE_MLOCK		/* mlock() is present in libc */
-#undef HAVE_MLOCKALL		/* working mlockall() is present in libc */
-#undef HAVE_MMAP		/* working mmap() is present in libc */
-#undef _MMAP_WITH_SIZEP		/* mmap() needs address of size parameter */
-#undef HAVE_FLOCK		/* *BSD flock() is present in libc */
-#undef HAVE_LOCKF		/* lockf() is present in libc (XOPEN) */
-#undef HAVE_FCNTL_LOCKF		/* file locking via fcntl() is present in libc */
-#undef HAVE_FCHDIR		/* fchdir() is present in libc */
-#undef HAVE_STATVFS		/* statvfs() is present in libc */
-#undef HAVE_QUOTACTL		/* quotactl() is present in libc */
-#undef HAVE_QUOTAIOCTL		/* use ioctl(f, Q_QUOTACTL, &q) instead of quotactl() */
-#undef HAVE_SETREUID		/* setreuid() is present in libc */
-#undef HAVE_SETRESUID		/* setresuid() is present in libc */
-#undef HAVE_SETEUID		/* seteuid() is present in libc */
-#undef HAVE_SETUID		/* setuid() is present in libc */
-#undef HAVE_SETREGID		/* setregid() is present in libc */
-#undef HAVE_SETRESGID		/* setresgid() is present in libc */
-#undef HAVE_SETEGID		/* setegid() is present in libc */
-#undef HAVE_SETGID		/* setgid() is present in libc */
-#undef HAVE_GETPGID		/* getpgid() is present in libc (POSIX) */
-#undef HAVE_SETPGID		/* setpgid() is present in libc (POSIX) */
-#undef HAVE_GETPGRP		/* getpgrp() is present in libc (ANY) */
-#undef HAVE_SETPGRP		/* setpgrp() is present in libc (ANY) */
-#undef HAVE_BSD_GETPGRP		/* getpgrp() in libc is BSD-4.2 compliant */
-#undef HAVE_BSD_SETPGRP		/* setpgrp() in libc is BSD-4.2 compliant */
-#undef HAVE_GETSPNAM		/* getspnam() in libc (SVR4 compliant) */
-#undef HAVE_GETSPWNAM		/* getspwnam() in libsec.a (HP-UX) */
-#undef HAVE_SYNC		/* sync() is present in libc */
-#undef HAVE_FSYNC		/* fsync() is present in libc */
-#undef HAVE_WAIT3		/* working wait3() is present in libc */
-#undef HAVE_WAIT4		/* wait4() is present in libc */
-#undef HAVE_WAITID		/* waitid() is present in libc */
-#undef HAVE_WAITPID		/* waitpid() is present in libc */
-#undef HAVE_GETHOSTID		/* gethostid() is present in libc */
-#undef HAVE_GETHOSTNAME		/* gethostname() is present in libc */
-#undef HAVE_GETDOMAINNAME	/* getdomainname() is present in libc */
-#undef HAVE_GETPAGESIZE		/* getpagesize() is present in libc */
-#undef HAVE_GETDTABLESIZE	/* getdtablesize() is present in libc */
-#undef HAVE_GETRUSAGE		/* getrusage() is present in libc */
-#undef HAVE_GETRLIMIT		/* getrlimit() is present in libc */
-#undef HAVE_SETRLIMIT		/* setrlimit() is present in libc */
-#undef HAVE_ULIMIT		/* ulimit() is present in libc */
-#undef HAVE_GETTIMEOFDAY	/* gettimeofday() is present in libc */
-#undef HAVE_SETTIMEOFDAY	/* settimeofday() is present in libc */
-#undef HAVE_TIME		/* time() is present in libc */
-#undef HAVE_STIME		/* stime() is present in libc */
-#undef HAVE_POLL		/* poll() is present in libc */
-#undef HAVE_SELECT		/* select() is present in libc */
-#undef HAVE_LCHOWN		/* lchown() is present in libc */
-#undef HAVE_BRK			/* brk() is present in libc */
-#undef HAVE_SBRK		/* sbrk() is present in libc */
-#undef HAVE_VA_COPY		/* va_copy() is present in varargs.h/stdarg.h */
-#undef HAVE__VA_COPY		/* __va_copy() is present in varargs.h/stdarg.h */
-#undef HAVE_DTOA		/* BSD-4.4 __dtoa() is present in libc */
-#undef HAVE_DTOA_R		/* BSD-4.4 __dtoa() with result ptr (reentrant) */
-#undef HAVE_GETCWD		/* POSIX getcwd() is present in libc */
-#undef HAVE_SMMAP		/* may map anonymous memory to get shared mem */
-#undef HAVE_SHMAT		/* shmat() is present in libc */
-#undef HAVE_SEMGET		/* semget() is present in libc */
-#undef HAVE_LSTAT		/* lstat() is present in libc */
-#undef HAVE_READLINK		/* readlink() is present in libc */
-#undef HAVE_LINK		/* link() is present in libc */
-#undef HAVE_RENAME		/* rename() is present in libc */
-#undef HAVE_MKFIFO		/* mkfifo() is present in libc */
-#undef HAVE_MKNOD		/* mknod() is present in libc */
-#undef HAVE_ECVT		/* ecvt() is present in libc */
-#undef HAVE_FCVT		/* fcvt() is present in libc */
-#undef HAVE_GCVT		/* gcvt() is present in libc */
-#undef HAVE_ECVT_R		/* ecvt_r() is present in libc */
-#undef HAVE_FCVT_R		/* fcvt_r() is present in libc */
-#undef HAVE_GCVT_R		/* gcvt_r() is present in libc */
-#undef HAVE_ECONVERT		/* econvert() is present in libc */
-#undef HAVE_FCONVERT		/* fconvert() is present in libc */
-#undef HAVE_GCONVERT		/* gconvert() is present in libc */
-#undef HAVE_ISINF		/* isinf() is present in libc */
-#undef HAVE_ISNAN		/* isnan() is present in libc */
-#undef HAVE_RAND		/* rand() is present in libc */
-#undef HAVE_DRAND48		/* drand48() is present in libc */
-#undef HAVE_SETPRIORITY		/* setpriority() is present in libc */
-#undef HAVE_NICE		/* nice() is present in libc */
-#undef HAVE_DOSSETPRIORITY	/* DosSetPriority() is present in libc */
-#undef HAVE_DOSALLOCSHAREDMEM	/* DosAllocSharedMem() is present in libc */
-#undef HAVE_SEEKDIR		/* seekdir() is present in libc */
-#undef HAVE_PUTENV		/* putenv() is present in libc (preferred function) */
-#undef HAVE_SETENV		/* setenv() is present in libc (use instead of putenv()) */
-#undef HAVE_UNAME		/* uname() is present in libc */
-#undef HAVE_SNPRINTF		/* snprintf() is present in libc */
-#undef HAVE_STRCASECMP		/* strcasecmp() is present in libc */
-#undef HAVE_STRDUP		/* strdup() is present in libc */
-#undef HAVE_STRSIGNAL		/* strsignal() is present in libc */
-#undef HAVE_STR2SIG		/* str2sig() is present in libc */
-#undef HAVE_SIG2STR		/* sig2str() is present in libc */
-#undef HAVE_KILLPG		/* killpg() is present in libc */
-#undef HAVE_SIGRELSE		/* sigrelse() is present in libc */
-#undef HAVE_SIGPROCMASK		/* sigprocmask() is present in libc (POSIX) */
-#undef HAVE_SIGSETMASK		/* sigsetmask() is present in libc (BSD) */
-#undef HAVE_SIGSET		/* sigset() is present in libc (POSIX) */
-#undef HAVE_SYS_SIGLIST		/* char *sys_siglist[] is present in libc */
-#undef HAVE_NANOSLEEP		/* nanosleep() is present in libc */
-#undef HAVE_USLEEP		/* usleep() is present in libc */
-#undef HAVE_FORK		/* fork() is present in libc */
-#undef HAVE_EXECL		/* execl() is present in libc */
-#undef HAVE_EXECLE		/* execle() is present in libc */
-#undef HAVE_EXECV		/* execv() is present in libc */
-#undef HAVE_EXECVE		/* execve() is present in libc */
-#undef HAVE_EXECVP		/* execvp() is present in libc */
-#undef HAVE_VALLOC		/* valloc() is present in libc (else use malloc())*/
+#undef HAVE_STRERROR
+#undef HAVE_MEMMOVE
+#undef HAVE_MADVISE
+#undef HAVE_MLOCK
+#undef HAVE_MLOCKALL
+#undef HAVE_MMAP
+#undef _MMAP_WITH_SIZEP
+#undef HAVE_FLOCK
+#undef HAVE_LOCKF
+#undef HAVE_FCNTL_LOCKF
+#undef HAVE_FCHDIR
+#undef HAVE_STATVFS
+#undef HAVE_QUOTACTL
+#undef HAVE_QUOTAIOCTL
+#undef HAVE_SETREUID
+#undef HAVE_SETRESUID
+#undef HAVE_SETEUID
+#undef HAVE_SETUID
+#undef HAVE_SETREGID
+#undef HAVE_SETRESGID
+#undef HAVE_SETEGID
+#undef HAVE_SETGID
+#undef HAVE_GETPGID
+#undef HAVE_SETPGID
+#undef HAVE_GETPGRP
+#undef HAVE_SETPGRP
+#undef HAVE_BSD_GETPGRP
+#undef HAVE_BSD_SETPGRP
+#undef HAVE_GETSPNAM
+#undef HAVE_GETSPWNAM
+#undef HAVE_SYNC
+#undef HAVE_FSYNC
+#undef HAVE_WAIT3
+#undef HAVE_WAIT4
+#undef HAVE_WAITID
+#undef HAVE_WAITPID
+#undef HAVE_GETHOSTID
+#undef HAVE_GETHOSTNAME
+#undef HAVE_GETDOMAINNAME
+#undef HAVE_GETPAGESIZE
+#undef HAVE_GETDTABLESIZE
+#undef HAVE_GETRUSAGE
+#undef HAVE_GETRLIMIT
+#undef HAVE_SETRLIMIT
+#undef HAVE_ULIMIT
+#undef HAVE_GETTIMEOFDAY
+#undef HAVE_SETTIMEOFDAY
+#undef HAVE_TIME
+#undef HAVE_STIME
+#undef HAVE_POLL
+#undef HAVE_SELECT
+#undef HAVE_LCHOWN
+#undef HAVE_BRK
+#undef HAVE_SBRK
+#undef HAVE_VA_COPY
+#undef HAVE__VA_COPY
+#undef HAVE_DTOA
+#undef HAVE_DTOA_R
+#undef HAVE_GETCWD
+#undef HAVE_SMMAP
+#undef HAVE_SHMAT
+#undef HAVE_SEMGET
+#undef HAVE_LSTAT
+#undef HAVE_READLINK
+#undef HAVE_LINK
+#undef HAVE_RENAME
+#undef HAVE_MKFIFO
+#undef HAVE_MKNOD
+#undef HAVE_ECVT
+#undef HAVE_FCVT
+#undef HAVE_GCVT
+#undef HAVE_ECVT_R
+#undef HAVE_FCVT_R
+#undef HAVE_GCVT_R
+#undef HAVE_ECONVERT
+#undef HAVE_FCONVERT
+#undef HAVE_GCONVERT
+#undef HAVE_ISINF
+#undef HAVE_ISNAN
+#undef HAVE_RAND
+#undef HAVE_DRAND48
+#undef HAVE_SETPRIORITY
+#undef HAVE_NICE
+#undef HAVE_DOSSETPRIORITY
+#undef HAVE_DOSALLOCSHAREDMEM
+#undef HAVE_SEEKDIR
+#undef HAVE_PUTENV
+#undef HAVE_SETENV
+#undef HAVE_UNAME
+#undef HAVE_SNPRINTF
+#undef HAVE_STRCASECMP
+#undef HAVE_STRDUP
+#undef HAVE_STRSIGNAL
+#undef HAVE_STR2SIG
+#undef HAVE_SIG2STR
+#undef HAVE_KILLPG
+#undef HAVE_SIGRELSE
+#undef HAVE_SIGPROCMASK
+#undef HAVE_SIGSETMASK
+#undef HAVE_SIGSET
+#undef HAVE_SYS_SIGLIST
+#undef HAVE_NANOSLEEP
+#undef HAVE_USLEEP
+#undef HAVE_FORK
+#undef HAVE_EXECL
+#undef HAVE_EXECLE
+#undef HAVE_EXECV
+#undef HAVE_EXECVE
+#undef HAVE_EXECVP
+#undef HAVE_VALLOC
 #undef vfork
 
-#undef HAVE_CHFLAGS		/* chflags() is present in libc */
-#undef HAVE_FCHFLAGS		/* fchflags() is present in libc */
-#undef HAVE_FFLAGSTOSTR		/* fflagstostr() is present in libc */
-#undef HAVE_STRTOFFLAGS		/* strtofflags() is present in libc */
+#undef HAVE_CHFLAGS
+#undef HAVE_FCHFLAGS
+#undef HAVE_FFLAGSTOSTR
+#undef HAVE_STRTOFFLAGS
 
 /*
  * Important:	This must be a result from a check _before_ the Large File test
@@ -249,14 +249,14 @@
  *	Instead use the tests AC_SMALL_FSEEKO/AC_SMALL/STELLO and make sure
  *	they are placed before the large file tests.
  */
-#undef HAVE_FSEEKO		/* fseeko() is present in default compile mode */
-#undef HAVE_FTELLO		/* ftello() is present in default compile mode */
+#undef HAVE_FSEEKO
+#undef HAVE_FTELLO
 
-#undef HAVE_RCMD		/* rcmd() is present in libc/libsocket */
-#undef HAVE_INET_NTOA		/* inet_ntoa() is present in libc/libsocket */
+#undef HAVE_RCMD
+#undef HAVE_INET_NTOA
 
 #if	defined(HAVE_QUOTACTL) || defined(HAVE_QUOTAIOCTL)
-#	define HAVE_QUOTA	/* The system inludes quota */
+#	define HAVE_QUOTA
 #endif
 
 /*
@@ -265,52 +265,52 @@
  * from the BSD compile environment.
  */
 #if	defined(HAVE_SHMAT) && defined(HAVE_SYS_SHM_H) && defined(HAVE_SYS_IPC_H)
-#	define	HAVE_USGSHM	/* USG shared memory is present */
+#	define	HAVE_USGSHM
 #endif
 #if	defined(HAVE_SEMGET) && defined(HAVE_SYS_SHM_H) && defined(HAVE_SYS_IPC_H)
-#	define	HAVE_USGSEM	/* USG semaphores are present */
+#	define	HAVE_USGSEM
 #endif
 
 #if	defined(HAVE_GETPGRP) && !defined(HAVE_BSD_GETPGRP)
-#define	HAVE_POSIX_GETPGRP 1	/* getpgrp() in libc is POSIX compliant */
+#define	HAVE_POSIX_GETPGRP 1
 #endif
 #if	defined(HAVE_SETPGRP) && !defined(HAVE_BSD_SETPGRP)
-#define	HAVE_POSIX_SETPGRP 1	/* setpgrp() in libc is POSIX compliant */
+#define	HAVE_POSIX_SETPGRP 1
 #endif
 
 /*
  * Structures
  */
-#undef HAVE_MTGET_TYPE		/* if struct mtget contains mt_type (drive type) */
-#undef HAVE_MTGET_DSREG		/* if struct mtget contains mt_dsreg (drive status) */
-#undef HAVE_MTGET_ERREG		/* if struct mtget contains mt_erreg (error register) */
-#undef HAVE_MTGET_RESID		/* if struct mtget contains mt_resid (residual count) */
-#undef HAVE_MTGET_FILENO	/* if struct mtget contains mt_fileno (file #) */
-#undef HAVE_MTGET_BLKNO		/* if struct mtget contains mt_blkno (block #) */
-#undef HAVE_MTGET_FLAGS		/* if struct mtget contains mt_flags (flags) */
-#undef HAVE_MTGET_BF		/* if struct mtget contains mt_bf (optimum blocking factor) */
-#undef HAVE_STRUCT_RUSAGE	/* have struct rusage in sys/resource.h */
-#undef HAVE_UNION_SEMUN		/* have an illegal definition for union semun in sys/sem.h */
-#undef HAVE_UNION_WAIT		/* have union wait in wait.h */
-#undef HAVE_ST_SPARE1		/* if struct stat contains st_spare1 (usecs) */
-#undef HAVE_ST_NSEC		/* if struct stat contains st_atim.st_nsec (nanosecs */
-#undef HAVE_ST_BLKSIZE		/* if struct stat contains st_blksize */
-#undef HAVE_ST_BLOCKS		/* if struct stat contains st_blocks */
-#undef HAVE_ST_RDEV		/* if struct stat contains st_rdev */
-#undef HAVE_ST_FLAGS		/* if struct stat contains st_flags */
-#undef STAT_MACROS_BROKEN	/* if the macros S_ISDIR, S_ISREG .. don't work */
+#undef HAVE_MTGET_TYPE
+#undef HAVE_MTGET_DSREG
+#undef HAVE_MTGET_ERREG
+#undef HAVE_MTGET_RESID
+#undef HAVE_MTGET_FILENO
+#undef HAVE_MTGET_BLKNO
+#undef HAVE_MTGET_FLAGS
+#undef HAVE_MTGET_BF
+#undef HAVE_STRUCT_RUSAGE
+#undef HAVE_UNION_SEMUN
+#undef HAVE_UNION_WAIT
+#undef HAVE_ST_SPARE1
+#undef HAVE_ST_NSEC
+#undef HAVE_ST_BLKSIZE
+#undef HAVE_ST_BLOCKS
+#undef HAVE_ST_RDEV
+#undef HAVE_ST_FLAGS
+#undef STAT_MACROS_BROKEN
 
-#undef DEV_MINOR_BITS		/* # if bits needed to hold minor device number */
-#undef DEV_MINOR_NONCONTIG	/* if bits in minor device number are noncontiguous */
+#undef DEV_MINOR_BITS
+#undef DEV_MINOR_NONCONTIG
 
 
 /*
  * Byteorder/Bitorder
  */
-#define	HAVE_C_BIGENDIAN	/* Flag that WORDS_BIGENDIAN test was done */
-#undef WORDS_BIGENDIAN		/* If using network byte order             */
-#define	HAVE_C_BITFIELDS	/* Flag that BITFIELDS_HTOL test was done  */
-#undef BITFIELDS_HTOL		/* If high bits come first in structures   */
+#define	HAVE_C_BIGENDIAN
+#undef WORDS_BIGENDIAN
+#define	HAVE_C_BITFIELDS
+#undef BITFIELDS_HTOL
 
 /*
  * Types/Keywords
@@ -328,24 +328,24 @@
 #undef SIZEOF_UNSIGNED_LONG_LONG
 #undef SIZEOF_UNSIGNED_CHAR_P
 
-#undef HAVE_LONGLONG		/* Compiler defines long long type */
-#undef CHAR_IS_UNSIGNED		/* Compiler defines char to be unsigned */
+#undef HAVE_LONGLONG
+#undef CHAR_IS_UNSIGNED
 
-#undef const			/* Define to empty if const doesn't work */
-#undef uid_t			/* To be used if uid_t is not present  */
-#undef gid_t			/* To be used if gid_t is not present  */
-#undef size_t			/* To be used if size_t is not present */
-#undef ssize_t			/* To be used if ssize_t is not present */
-#undef pid_t			/* To be used if pid_t is not present  */
-#undef off_t			/* To be used if off_t is not present  */
-#undef mode_t			/* To be used if mode_t is not present */
-#undef time_t			/* To be used if time_t is not present */
-#undef caddr_t			/* To be used if caddr_t is not present */
-#undef daddr_t			/* To be used if daddr_t is not present */
-#undef dev_t			/* To be used if dev_t is not present */
-#undef ino_t			/* To be used if ino_t is not present */
-#undef nlink_t			/* To be used if nlink_t is not present */
-#undef socklen_t		/* To be used if socklen_t is not present */
+#undef const
+#undef uid_t
+#undef gid_t
+#undef size_t
+#undef ssize_t
+#undef pid_t
+#undef off_t
+#undef mode_t
+#undef time_t
+#undef caddr_t
+#undef daddr_t
+#undef dev_t
+#undef ino_t
+#undef nlink_t
+#undef socklen_t
 
 /*
  * These types are present on all UNIX systems but should be avoided
@@ -354,14 +354,14 @@
  *
  * Better include <utypes.h> and use Uchar, Uint & Ulong
  */
-#undef u_char			/* To be used if u_char is not present	*/
-#undef u_short			/* To be used if u_short is not present	*/
-#undef u_int			/* To be used if u_int is not present	*/
-#undef u_long			/* To be used if u_long is not present	*/
+#undef u_char
+#undef u_short
+#undef u_int
+#undef u_long
 
 /*#undef HAVE_SIZE_T*/
 /*#undef NO_SIZE_T*/
-#undef VA_LIST_IS_ARRAY		/* va_list is an array */
+#undef VA_LIST_IS_ARRAY
 #undef GETGROUPS_T
 #define GID_T		GETGROUPS_T
 
@@ -377,74 +377,74 @@
 
 #undef	HAVE_LARGEFILES
 
-#ifdef	HAVE_LARGEFILES		/* If we have working largefiles at all	   */
+#ifdef	HAVE_LARGEFILES
 				/* This is not defined with glibc-2.1.3	   */
 
-#undef _FILE_OFFSET_BITS	/* # of bits in off_t if settable	   */
-#undef _LARGEFILE_SOURCE	/* To make ftello() visible (HP-UX 10.20). */
-#undef _LARGE_FILES		/* Large file defined on AIX-style hosts.  */
-#undef _XOPEN_SOURCE		/* To make ftello() visible (glibc 2.1.3). */
+#undef _FILE_OFFSET_BITS
+#undef _LARGEFILE_SOURCE
+#undef _LARGE_FILES
+#undef _XOPEN_SOURCE
 				/* XXX We don't use this because glibc2.1.3*/
 				/* XXX is bad anyway. If we define	   */
 				/* XXX _XOPEN_SOURCE we will loose caddr_t */
 
-#undef HAVE_FSEEKO		/* Do we need this? If HAVE_LARGEFILES is  */
+#undef HAVE_FSEEKO
 				/* defined, we have fseeko()		   */
 
 #endif	/* HAVE_LARGEFILES */
 #endif	/* USE_LARGEFILES */
 
-#ifdef USE_ACL			/* Enable/disable ACL support */
+#ifdef USE_ACL
 /*
  * POSIX ACL support
  */
-#undef HAVE_ACL_GET_FILE	/* acl_get_file() function */
-#undef HAVE_ACL_SET_FILE	/* acl_set_file() function */
-#undef HAVE_ACL_FROM_TEXT	/* acl_from_text() function */
-#undef HAVE_ACL_TO_TEXT		/* acl_to_text() function */
-#undef HAVE_ACL_FREE		/* acl_free() function */
-#undef HAVE_ACL_DELETE_DEF_FILE	/* acl_delete_def_file() function */
+#undef HAVE_ACL_GET_FILE
+#undef HAVE_ACL_SET_FILE
+#undef HAVE_ACL_FROM_TEXT
+#undef HAVE_ACL_TO_TEXT
+#undef HAVE_ACL_FREE
+#undef HAVE_ACL_DELETE_DEF_FILE
 
 #if defined(HAVE_ACL_GET_FILE) && defined(HAVE_ACL_SET_FILE) && \
     defined(HAVE_ACL_FROM_TEXT) && defined(HAVE_ACL_TO_TEXT) && \
     defined(HAVE_ACL_FREE)
-#	define	HAVE_POSIX_ACL	1 /* POSIX ACL's present */
+#	define	HAVE_POSIX_ACL	1
 #endif
 
 /*
  * Sun ACL support.
  * Note: unfortunately, HP-UX has an (undocumented) acl() function in libc.
  */
-#undef HAVE_ACL			/* acl() function */
-#undef HAVE_FACL		/* facl() function */
-#undef HAVE_ACLFROMTEXT		/* aclfromtext() function */
-#undef HAVE_ACLTOTEXT		/* acltotext() function */
+#undef HAVE_ACL
+#undef HAVE_FACL
+#undef HAVE_ACLFROMTEXT
+#undef HAVE_ACLTOTEXT
 
 #if defined(HAVE_ACL) && defined(HAVE_FACL) && \
     defined(HAVE_ACLFROMTEXT) && defined(HAVE_ACLTOTEXT)
-#	define	HAVE_SUN_ACL	1 /* Sun ACL's present */
+#	define	HAVE_SUN_ACL	1
 #endif
 
 /*
  * HP-UX ACL support.
  * Note: unfortunately, HP-UX has an (undocumented) acl() function in libc.
  */
-#undef HAVE_GETACL		/* getacl() function */
-#undef HAVE_FGETACL		/* fgetacl() function */
-#undef HAVE_SETACL		/* setacl() function */
-#undef HAVE_FSETACL		/* fsetacl() function */
-#undef HAVE_STRTOACL		/* strtoacl() function */
-#undef HAVE_ACLTOSTR		/* acltostr() function */
-#undef HAVE_CPACL		/* cpacl() function */
-#undef HAVE_FCPACL		/* fcpacl() function */
-#undef HAVE_CHOWNACL		/* chownacl() function */
-#undef HAVE_SETACLENTRY		/* setaclentry() function */
-#undef HAVE_FSETACLENTRY	/* fsetaclentry() function */
+#undef HAVE_GETACL
+#undef HAVE_FGETACL
+#undef HAVE_SETACL
+#undef HAVE_FSETACL
+#undef HAVE_STRTOACL
+#undef HAVE_ACLTOSTR
+#undef HAVE_CPACL
+#undef HAVE_FCPACL
+#undef HAVE_CHOWNACL
+#undef HAVE_SETACLENTRY
+#undef HAVE_FSETACLENTRY
 
 #if defined(HAVE_GETACL) && defined(HAVE_FGETACL) && \
     defined(HAVE_SETACL) && defined(HAVE_FSETACL) && \
     defined(HAVE_STRTOACL) && defined(HAVE_ACLTOTEXT)
-#	define	HAVE_HP_ACL	1 /* HP-UX ACL's present */
+#	define	HAVE_HP_ACL	1
 #endif
 
 /*
@@ -453,7 +453,7 @@
  * included in HAVE_ANY_ACL.
  */
 #if defined(HAVE_POSIX_ACL) || defined(HAVE_SUN_ACL)
-#	define	HAVE_ANY_ACL	1 /* Any ACL implementation present */
+#	define	HAVE_ANY_ACL	1
 #endif
 
 #endif	/* USE_ACL */
@@ -461,16 +461,16 @@
 /*
  * Misc CC / LD related stuff
  */
-#undef NO_USER_MALLOC		/* If we cannot define our own malloc()	*/
+#undef NO_USER_MALLOC
 
 /*
  * Strings that help to maintain OS/platform id's in C-programs
  */
-#undef HOST_ALIAS		/* Output from config.guess (orig)	*/
-#undef HOST_SUB			/* Output from config.sub (modified)	*/
-#undef HOST_CPU			/* CPU part from HOST_SUB		*/
-#undef HOST_VENDOR		/* VENDOR part from HOST_SUB		*/
-#undef HOST_OS			/* CPU part from HOST_SUB		*/
+#undef HOST_ALIAS
+#undef HOST_SUB
+#undef HOST_CPU
+#undef HOST_VENDOR
+#undef HOST_OS
 
 
 /*
--- cdrtools-1.11/conf/aclocal.m4.orig	Sat Mar 23 21:39:26 2002
+++ cdrtools-1.11/conf/aclocal.m4	Wed Apr 24 22:39:46 2002
@@ -17,7 +17,7 @@
 
 dnl Checks if structure 'stat' have field 'st_spare1'.
 dnl Defines HAVE_ST_SPARE1 on success.
-AC_DEFUN(AC_STRUCT_ST_SPARE1,
+AC_DEFUN([AC_STRUCT_ST_SPARE1],
 [AC_CACHE_CHECK([if struct stat contains st_spare1], ac_cv_struct_st_spare1,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/stat.h>],
@@ -30,7 +30,7 @@
 
 dnl Checks if structure 'stat' have field 'st_atim.tv_nsec'.
 dnl Defines HAVE_ST_NSEC on success.
-AC_DEFUN(AC_STRUCT_ST_NSEC,
+AC_DEFUN([AC_STRUCT_ST_NSEC],
 [AC_CACHE_CHECK([if struct stat contains st_atim.tv_nsec], ac_cv_struct_st_nsec,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/stat.h>],
@@ -43,7 +43,7 @@
 
 dnl Checks if structure 'stat' have field 'st_flags'.
 dnl Defines HAVE_ST_FLAGS on success.
-AC_DEFUN(AC_STRUCT_ST_FLAGS,
+AC_DEFUN([AC_STRUCT_ST_FLAGS],
 [AC_CACHE_CHECK([if struct stat contains st_flags], ac_cv_struct_st_flags,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/stat.h>],
@@ -56,7 +56,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_type'.
 dnl Defines HAVE_MTGET_TYPE on success.
-AC_DEFUN(AC_STRUCT_MTGET_TYPE,
+AC_DEFUN([AC_STRUCT_MTGET_TYPE],
 [AC_CACHE_CHECK([if struct mtget contains mt_type], ac_cv_struct_mtget_type,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -69,7 +69,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_dsreg'.
 dnl Defines HAVE_MTGET_DSREG on success.
-AC_DEFUN(AC_STRUCT_MTGET_DSREG,
+AC_DEFUN([AC_STRUCT_MTGET_DSREG],
 [AC_CACHE_CHECK([if struct mtget contains mt_dsreg], ac_cv_struct_mtget_dsreg,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -82,7 +82,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_erreg'.
 dnl Defines HAVE_MTGET_ERREG on success.
-AC_DEFUN(AC_STRUCT_MTGET_ERREG,
+AC_DEFUN([AC_STRUCT_MTGET_ERREG],
 [AC_CACHE_CHECK([if struct mtget contains mt_erreg], ac_cv_struct_mtget_erreg,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -95,7 +95,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_resid'.
 dnl Defines HAVE_MTGET_RESID on success.
-AC_DEFUN(AC_STRUCT_MTGET_RESID,
+AC_DEFUN([AC_STRUCT_MTGET_RESID],
 [AC_CACHE_CHECK([if struct mtget contains mt_resid], ac_cv_struct_mtget_resid,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -108,7 +108,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_fileno'.
 dnl Defines HAVE_MTGET_FILENO on success.
-AC_DEFUN(AC_STRUCT_MTGET_FILENO,
+AC_DEFUN([AC_STRUCT_MTGET_FILENO],
 [AC_CACHE_CHECK([if struct mtget contains mt_fileno],
                 ac_cv_struct_mtget_fileno,
                 [AC_TRY_COMPILE([#include <sys/types.h>
@@ -122,7 +122,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_blkno'.
 dnl Defines HAVE_MTGET_BLKNO on success.
-AC_DEFUN(AC_STRUCT_MTGET_BLKNO,
+AC_DEFUN([AC_STRUCT_MTGET_BLKNO],
 [AC_CACHE_CHECK([if struct mtget contains mt_blkno], ac_cv_struct_mtget_blkno,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -135,7 +135,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_flags'.
 dnl Defines HAVE_MTGET_FLAGS on success.
-AC_DEFUN(AC_STRUCT_MTGET_FLAGS,
+AC_DEFUN([AC_STRUCT_MTGET_FLAGS],
 [AC_CACHE_CHECK([if struct mtget contains mt_flags], ac_cv_struct_mtget_flags,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -148,7 +148,7 @@
 
 dnl Checks if structure 'mtget' have field 'mt_bf'.
 dnl Defines HAVE_MTGET_BF on success.
-AC_DEFUN(AC_STRUCT_MTGET_BF,
+AC_DEFUN([AC_STRUCT_MTGET_BF],
 [AC_CACHE_CHECK([if struct mtget contains mt_bf], ac_cv_struct_mtget_bf,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/mtio.h>],
@@ -161,7 +161,7 @@
 
 dnl Checks for illegal declaration of 'union semun' in sys/sem.h.
 dnl Defines HAVE_UNION_SEMUN on success.
-AC_DEFUN(AC_STRUCT_UNION_SEMUN,
+AC_DEFUN([AC_STRUCT_UNION_SEMUN],
 [AC_CACHE_CHECK([if an illegal declaration for union semun in sys/sem.h exists], ac_cv_struct_union_semun,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/ipc.h>
@@ -174,7 +174,7 @@
 
 dnl Checks if 'union wait' is declared in 'wait.h' or 'sys/wait.h'.
 dnl Defines HAVE_UNION_WAIT on success.
-AC_DEFUN(AC_STRUCT_UNION_WAIT,
+AC_DEFUN([AC_STRUCT_UNION_WAIT],
 [AC_CACHE_CHECK([if union wait is declared in wait.h or sys/wait.h], ac_cv_struct_union_wait,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #if	defined(HAVE_WAIT_H)
@@ -190,7 +190,7 @@
 
 dnl Checks if 'struct rusage' is declared in sys/resource.h.
 dnl Defines HAVE_STRUCT_RUSAGE on success.
-AC_DEFUN(AC_STRUCT_RUSAGE,
+AC_DEFUN([AC_STRUCT_RUSAGE],
 [AC_CACHE_CHECK([if struct rusage is declared in sys/resource.h], ac_cv_struct_rusage,
                 [AC_TRY_COMPILE([#include <sys/time.h>
 #include <sys/resource.h>], [struct rusage r;],
@@ -203,7 +203,7 @@
 dnl Checks wether major(), minor() and makedev() are defined in
 dnl 'sys/mkdev.h' or in 'sys/sysmacros.h. Defines MAJOR_IN_MKDEV or
 dnl MAJOR_IN_SYSMACROS or nothing.
-AC_DEFUN(AC_HEADER_MAKEDEV,
+AC_DEFUN([AC_HEADER_MAKEDEV],
 [AC_CACHE_CHECK([for header file containing  major(), minor() and makedev()],
                ac_cv_header_makedev,
 [ac_cv_header_makedev=none
@@ -226,7 +226,7 @@
 
 dnl Checks for USG derived STDIO that uses _filbuf()
 dnl Defines HAVE__FILBUF on success.
-AC_DEFUN(AC_HEADER__FILBUF,
+AC_DEFUN([AC_HEADER__FILBUF],
 [AC_CACHE_CHECK([for _filbuf()], ac_cv_func__filbuf,
                 [AC_TRY_LINK([#include <stdio.h>],
 [FILE    *f;
@@ -251,7 +251,7 @@
 
 dnl Checks for USG derived STDIO that uses __filbuf()
 dnl Defines HAVE___FILBUF on success.
-AC_DEFUN(AC_HEADER___FILBUF,
+AC_DEFUN([AC_HEADER___FILBUF],
 [AC_CACHE_CHECK([for __filbuf()], ac_cv_func___filbuf,
                 [AC_TRY_LINK([#include <stdio.h>],
 [FILE    *f;
@@ -276,7 +276,7 @@
 
 dnl Checks for USG derived STDIO
 dnl Defines HAVE_USG_STDIO on success.
-AC_DEFUN(AC_HEADER_USG_STDIO,
+AC_DEFUN([AC_HEADER_USG_STDIO],
 [AC_REQUIRE([AC_HEADER__FILBUF])AC_REQUIRE([AC_HEADER___FILBUF])dnl
 AC_CACHE_CHECK([for USG derived STDIO], ac_cv_header_usg_stdio,
                 [AC_TRY_LINK([#include <stdio.h>],
@@ -311,7 +311,7 @@
 
 dnl Checks for errno definition in <errno.h>
 dnl Defines HAVE_ERRNO_DEF on success.
-AC_DEFUN(AC_HEADER_ERRNO_DEF,
+AC_DEFUN([AC_HEADER_ERRNO_DEF],
 [AC_CACHE_CHECK([for errno definition in errno.h], ac_cv_header_errno_def,
                 [AC_TRY_COMPILE([#include <errno.h>],
 [errno = 0;],
@@ -323,7 +323,7 @@
 
 dnl Checks for UNIX-98 compliant <inttypes.h>
 dnl Defines HAVE_INTTYPES_H on success.
-AC_DEFUN(AC_HEADER_INTTYPES,
+AC_DEFUN([AC_HEADER_INTTYPES],
 [AC_CACHE_CHECK([for UNIX-98 compliant inttypes.h], ac_cv_header_inttypes,
                 [AC_TRY_COMPILE([#include <inttypes.h>],
 [int8_t c; uint8_t uc; int16_t s; uint16_t us; int32_t i; uint32_t ui;
@@ -337,7 +337,7 @@
 
 dnl Checks for type time_t
 dnl Defines time_t to long on failure.
-AC_DEFUN(AC_TYPE_TIME_T,
+AC_DEFUN([AC_TYPE_TIME_T],
 [AC_REQUIRE([AC_HEADER_TIME])dnl
 AC_CACHE_CHECK([for time_t], ac_cv_type_time_t,
                 [AC_TRY_COMPILE([
@@ -360,7 +360,7 @@
 
 dnl Checks for type socklen_t
 dnl Defines socklen_t to int on failure.
-AC_DEFUN(AC_TYPE_SOCKLEN_T,
+AC_DEFUN([AC_TYPE_SOCKLEN_T],
 [AC_REQUIRE([AC_HEADER_STDC])dnl
 AC_MSG_CHECKING(for socklen_t)
 AC_CACHE_VAL(ac_cv_type_socklen_t,
@@ -380,7 +380,7 @@
 
 dnl Checks for type long long
 dnl Defines HAVE_LONGLONG on success.
-AC_DEFUN(AC_TYPE_LONGLONG,
+AC_DEFUN([AC_TYPE_LONGLONG],
 [AC_CACHE_CHECK([for type long long], ac_cv_type_longlong,
                 [AC_TRY_COMPILE([], [long long i;],
                 [ac_cv_type_longlong=yes],
@@ -391,7 +391,7 @@
 
 dnl Checks if C-compiler orders bitfields htol
 dnl Defines BITFIELDS_HTOL on success.
-AC_DEFUN(AC_C_BITFIELDS,
+AC_DEFUN([AC_C_BITFIELDS],
 [AC_CACHE_CHECK([whether bitorder in bitfields is htol], ac_cv_c_bitfields_htol,
                 [AC_TRY_RUN([
 struct {
@@ -414,7 +414,7 @@
 
 dnl Checks if C-compiler understands prototypes
 dnl Defines PROTOTYPES on success.
-AC_DEFUN(AC_TYPE_PROTOTYPES,
+AC_DEFUN([AC_TYPE_PROTOTYPES],
 [AC_CACHE_CHECK([for prototypes], ac_cv_type_prototypes,
                 [AC_TRY_RUN([
 doit(int i, ...)
@@ -431,7 +431,7 @@
 
 dnl Checks for type size_t
 dnl Defines HAVE_SIZE_T_ on success.
-AC_DEFUN(AC_TYPE_SIZE_T_,
+AC_DEFUN([AC_TYPE_SIZE_T_],
 [AC_CACHE_CHECK([for type size_t], ac_cv_type_size_t_,
                 [AC_TRY_COMPILE([#include <sys/types.h>], [size_t s;],
                 [ac_cv_type_size_t_=yes],
@@ -444,7 +444,7 @@
 
 dnl Checks if type char is unsigned
 dnl Defines CHAR_IS_UNSIGNED on success.
-AC_DEFUN(AC_TYPE_CHAR,
+AC_DEFUN([AC_TYPE_CHAR],
 [AC_CACHE_CHECK([if char is unsigned], ac_cv_type_char_unsigned,
                 [AC_TRY_RUN([
 int
@@ -463,7 +463,7 @@
 
 dnl Checks if function/macro va_copy() is available
 dnl Defines HAVE_VA_COPY on success.
-AC_DEFUN(AC_FUNC_VA_COPY,
+AC_DEFUN([AC_FUNC_VA_COPY],
 [AC_CACHE_CHECK([for va_copy], ac_cv_func_va_copy,
                 [AC_TRY_LINK([
 #ifdef	HAVE_STDARG_H
@@ -483,7 +483,7 @@
 
 dnl Checks if function/macro __va_copy() is available
 dnl Defines HAVE__VA_COPY on success.
-AC_DEFUN(AC_FUNC__VA_COPY,
+AC_DEFUN([AC_FUNC__VA_COPY],
 [AC_CACHE_CHECK([for __va_copy], ac_cv_func__va_copy,
                 [AC_TRY_LINK([
 #ifdef	HAVE_STDARG_H
@@ -504,7 +504,7 @@
 
 dnl Checks if va_list is an array
 dnl Defines VA_LIST_IS_ARRAY on success.
-AC_DEFUN(AC_TYPE_VA_LIST,
+AC_DEFUN([AC_TYPE_VA_LIST],
 [AC_CACHE_CHECK([if va_list is an array], ac_cv_type_va_list_array,
                 [AC_TRY_LINK([
 #ifdef	HAVE_STDARG_H
@@ -525,7 +525,7 @@
 
 dnl Checks if quotactl is present as ioctl
 dnl Defines HAVE_QUOTAIOCTL on success.
-AC_DEFUN(AC_FUNC_QUOTAIOCTL,
+AC_DEFUN([AC_FUNC_QUOTAIOCTL],
 [AC_CACHE_CHECK([if quotactl is an ioctl], ac_cv_func_quotaioctl,
                 [AC_TRY_LINK([#include <sys/types.h>
 #include <sys/fs/ufs_quota.h>],
@@ -538,7 +538,7 @@
 
 dnl Checks if function __dtoa() is available
 dnl Defines HAVE_DTOA on success.
-AC_DEFUN(AC_FUNC_DTOA,
+AC_DEFUN([AC_FUNC_DTOA],
 [AC_CACHE_CHECK([for __dtoa], ac_cv_func_dtoa,
                 [AC_TRY_LINK([extern  char *__dtoa();], 
 [int decpt; int sign; char *ep; char *bp;
@@ -551,7 +551,7 @@
 
 dnl Checks if reentrant __dtoa() exists (needs a result prt)
 dnl Defines HAVE_DTOA_R on success.
-AC_DEFUN(AC_FUNC_DTOA_R,
+AC_DEFUN([AC_FUNC_DTOA_R],
 [AC_REQUIRE([AC_FUNC_DTOA])dnl
 AC_CACHE_CHECK([for __dtoa that needs result ptr], ac_cv_func_dtoa_r,
                 [AC_TRY_RUN([
@@ -580,7 +580,7 @@
 
 dnl Checks if working ecvt() exists
 dnl Defines HAVE_ECVT on success.
-AC_DEFUN(AC_FUNC_ECVT,
+AC_DEFUN([AC_FUNC_ECVT],
 [AC_CACHE_CHECK([for working ecvt() ], ac_cv_func_ecvt,
                 [AC_TRY_RUN([
 extern	char *ecvt();
@@ -607,7 +607,7 @@
 
 dnl Checks if working fcvt() exists
 dnl Defines HAVE_FCVT on success.
-AC_DEFUN(AC_FUNC_FCVT,
+AC_DEFUN([AC_FUNC_FCVT],
 [AC_CACHE_CHECK([for working fcvt() ], ac_cv_func_fcvt,
                 [AC_TRY_RUN([
 extern	char *fcvt();
@@ -634,7 +634,7 @@
 
 dnl Checks if working gcvt() exists
 dnl Defines HAVE_GCVT on success.
-AC_DEFUN(AC_FUNC_GCVT,
+AC_DEFUN([AC_FUNC_GCVT],
 [AC_CACHE_CHECK([for working gcvt() ], ac_cv_func_gcvt,
                 [AC_TRY_RUN([
 extern	char *gcvt();
@@ -661,7 +661,7 @@
 
 dnl Checks if function uname() is available
 dnl Defines HAVE_UNAME on success.
-AC_DEFUN(AC_FUNC_UNAME,
+AC_DEFUN([AC_FUNC_UNAME],
 [AC_CACHE_CHECK([for uname], ac_cv_func_uname,
                 [AC_TRY_LINK([#include <sys/utsname.h>], 
 [struct	utsname un;
@@ -675,7 +675,7 @@
 dnl Checks if function mlockall() is available
 dnl beware HP-UX 10.x it contains a bad mlockall() in libc
 dnl Defines HAVE_MLOCKALL on success.
-AC_DEFUN(AC_FUNC_MLOCKALL,
+AC_DEFUN([AC_FUNC_MLOCKALL],
 [AC_CACHE_CHECK([for mlockall], ac_cv_func_mlockall,
                 [AC_TRY_RUN([
 #include <sys/types.h>
@@ -699,7 +699,7 @@
   AC_DEFINE(HAVE_MLOCKALL)
 fi])
 
-AC_DEFUN(jsAC_FUNC_MMAP,
+AC_DEFUN([jsAC_FUNC_MMAP],
 [AC_REQUIRE([AC_MMAP_SIZEP])dnl
 AC_CHECK_HEADERS(unistd.h)
 AC_CHECK_FUNCS(getpagesize)
@@ -856,7 +856,7 @@
 fi
 ])
 
-AC_DEFUN(AC_MMAP_SIZEP,
+AC_DEFUN([AC_MMAP_SIZEP],
 [AC_CHECK_HEADERS(unistd.h)
 AC_CHECK_FUNCS(getpagesize)
 AC_CACHE_CHECK(for mmap that needs ptr to size, ac_cv_func_mmap_sizep,
@@ -1006,7 +1006,7 @@
 
 dnl Checks if mmap() works to get shared memory
 dnl Defines HAVE_SMMAP on success.
-AC_DEFUN(AC_FUNC_SMMAP,
+AC_DEFUN([AC_FUNC_SMMAP],
 [AC_CACHE_CHECK([if mmap works to get shared memory], ac_cv_func_smmap,
                 [AC_TRY_RUN([
 #include <sys/types.h>
@@ -1086,7 +1089,7 @@
 
 dnl Checks if sys_siglist[] exists
 dnl Defines HAVE_SYS_SIGLIST on success.
-AC_DEFUN(AC_FUNC_SYS_SIGLIST,
+AC_DEFUN([AC_FUNC_SYS_SIGLIST],
 [AC_CACHE_CHECK([for sys_siglist], ac_cv_func_sys_siglist,
                 [AC_TRY_RUN([
 int
@@ -1102,7 +1105,7 @@
 fi])
 
 dnl Checks for maximum number of bits in minor device number
-AC_DEFUN(AC_CHECK_MINOR_BITS,
+AC_DEFUN([AC_CHECK_MINOR_BITS],
 [AC_REQUIRE([AC_HEADER_MAKEDEV])dnl
 changequote(<<, >>)dnl
 define(<<AC_MACRO_NAME>>, DEV_MINOR_BITS)dnl
@@ -1169,7 +1172,7 @@
 
 dnl Checks for maximum number of bits in minor device numbers are non contiguous
 dnl Defines DEV_MINOR_NONCONTIG on success.
-AC_DEFUN(AC_CHECK_MINOR_NONCONTIG,
+AC_DEFUN([AC_CHECK_MINOR_NONCONTIG],
 [AC_REQUIRE([AC_HEADER_MAKEDEV])dnl
 AC_CACHE_CHECK([whether bits in minor device numbers are non contiguous], ac_cv_dev_minor_noncontig,
                 [AC_TRY_RUN([
@@ -1225,7 +1228,7 @@
 
 dnl Checks if we may not define our own malloc()
 dnl Defines NO_USER_MALLOC if we cannot.
-AC_DEFUN(AC_USER_MALLOC,
+AC_DEFUN([AC_USER_MALLOC],
 [AC_CACHE_CHECK([if we may not define our own malloc()], ac_cv_no_user_malloc,
                 [AC_TRY_LINK([
 char * malloc(x)
@@ -1242,7 +1245,7 @@
 
 dnl Checks if BSD-4.2 compliant getpgrp() exists
 dnl Defines HAVE_BSD_GETPGRP on success.
-AC_DEFUN(AC_FUNC_BSD_GETPGRP,
+AC_DEFUN([AC_FUNC_BSD_GETPGRP],
 [AC_CACHE_CHECK([for BSD compliant getpgrp], ac_cv_func_bsd_getpgrp,
                 [AC_TRY_RUN([
 int
@@ -1263,7 +1266,7 @@
 
 dnl Checks if BSD-4.2 compliant setpgrp() exists
 dnl Defines HAVE_BSD_SETPGRP on success.
-AC_DEFUN(AC_FUNC_BSD_SETPGRP,
+AC_DEFUN([AC_FUNC_BSD_SETPGRP],
 [AC_REQUIRE([AC_HEADER_ERRNO_DEF])dnl
 AC_CACHE_CHECK([for BSD compliant setpgrp], ac_cv_func_bsd_setpgrp,
                 [AC_TRY_RUN([
@@ -1288,7 +1291,7 @@
 
 dnl Checks if select() needs more than sys/time.h & sys/types.h
 dnl Defines SELECT_NONSTD_HDR on success.
-AC_DEFUN(AC_HEADER_SELECT_NONSTD,
+AC_DEFUN([AC_HEADER_SELECT_NONSTD],
 [AC_CACHE_CHECK([if select needs nonstd include files], ac_cv_header_slect_nonstd_hdr,
                 [AC_TRY_COMPILE([#include <sys/types.h>
 #include <sys/time.h>],
@@ -1301,7 +1304,7 @@
 
 dnl Checks if select() needs sys/select.h
 dnl Defines NEED_SYS_SELECT_H on success.
-AC_DEFUN(AC_HEADER_SYS_SELECT,
+AC_DEFUN([AC_HEADER_SYS_SELECT],
 [AC_REQUIRE([AC_HEADER_SELECT_NONSTD])dnl
 AC_CACHE_CHECK([if sys/select.h is needed for select], ac_cv_header_need_sys_select_h,
                 [AC_TRY_COMPILE([#include <sys/types.h>
@@ -1319,7 +1322,7 @@
 
 dnl Checks if select() needs sys/socket.h
 dnl Defines NEED_SYS_SOCKET_H on success.
-AC_DEFUN(AC_HEADER_SELECT2,
+AC_DEFUN([AC_HEADER_SELECT2],
 [AC_REQUIRE([AC_HEADER_SELECT_NONSTD])dnl
 AC_CACHE_CHECK([if sys/socket.h is needed for select], ac_cv_header_need_sys_socket_h,
                 [AC_TRY_COMPILE([#include <sys/types.h>
@@ -1337,7 +1340,7 @@
 
 dnl Checks if file locking via fcntl() is available
 dnl Defines HAVE_FCNTL_LOCKF on success.
-AC_DEFUN(AC_FUNC_FCNTL_LOCKF,
+AC_DEFUN([AC_FUNC_FCNTL_LOCKF],
 [AC_CACHE_CHECK([for file locking via fcntl], ac_cv_func_fcntl_lock,
                 [AC_TRY_LINK([
 #include <sys/types.h>
@@ -1366,107 +1369,9 @@
 
 #serial 18
 
-dnl By default, many hosts won't let programs access large files;
-dnl one must use special compiler options to get large-file access to work.
-dnl For more details about this brain damage please see:
-dnl http://www.sas.com/standards/large.file/x_open.20Mar96.html
-
-dnl Written by Paul Eggert <eggert@twinsun.com>.
-
-dnl Internal subroutine of AC_SYS_LARGEFILE.
-dnl AC_SYS_LARGEFILE_TEST_INCLUDES
-AC_DEFUN(AC_SYS_LARGEFILE_TEST_INCLUDES,
-  [[#include <sys/types.h>
-    /* Check that off_t can represent 2**63 - 1 correctly.
-       We can't simply "#define LARGE_OFF_T 9223372036854775807",
-       since some C++ compilers masquerading as C compilers
-       incorrectly reject 9223372036854775807.  */
-#   define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
-    int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
-			&& LARGE_OFF_T % 2147483647 == 1)
-		       ? 1 : -1];
-  ]])
-
-dnl Internal subroutine of AC_SYS_LARGEFILE.
-dnl AC_SYS_LARGEFILE_MACRO_VALUE(C-MACRO, VALUE, CACHE-VAR, COMMENT, INCLUDES, FUNCTION-BODY)
-AC_DEFUN(AC_SYS_LARGEFILE_MACRO_VALUE,
-  [AC_CACHE_CHECK([for $1 value needed for large files], $3,
-     [$3=no
-      AC_TRY_COMPILE([$5],
-	[$6], 
-	,
-	[AC_TRY_COMPILE([#define $1 $2]
-[$5]
-	   ,
-	   [$6],
-	   [$3=$2])])])
-   if test "[$]$3" != no; then
-     AC_DEFINE_UNQUOTED([$1], [$]$3, [$4])
-   fi])
-
-AC_DEFUN(AC_SYS_LARGEFILE,
-  [AC_ARG_ENABLE(largefile,
-     [  --disable-largefile     omit support for large files])
-   if test "$enable_largefile" != no; then
-
-     AC_CACHE_CHECK([for special C compiler options needed for large files],
-       ac_cv_sys_largefile_CC,
-       [ac_cv_sys_largefile_CC=no
-        largefile_cc_opt=""
-        if test "$GCC" != yes; then
-	  # IRIX 6.2 and later do not support large files by default,
-	  # so use the C compiler's -n32 option if that helps.
-	  AC_TRY_COMPILE(AC_SYS_LARGEFILE_TEST_INCLUDES, , ,
-	    [ac_save_CC="${CC-cc}"
-	     CC="$CC -n32"
-	     AC_TRY_COMPILE(AC_SYS_LARGEFILE_TEST_INCLUDES, ,
-	       ac_cv_sys_largefile_CC=' -n32')
-	     CC="$ac_save_CC"])
-        fi])
-     if test "$ac_cv_sys_largefile_CC" != no; then
-       CC="$CC$ac_cv_sys_largefile_CC"
-       largefile_cc_opt="$ac_cv_sys_largefile_CC"
-     fi
-
-     AC_SYS_LARGEFILE_MACRO_VALUE(_FILE_OFFSET_BITS, 64,
-       ac_cv_sys_file_offset_bits,
-       [Number of bits in a file offset, on hosts where this is settable.],
-       AC_SYS_LARGEFILE_TEST_INCLUDES)
-     AC_SYS_LARGEFILE_MACRO_VALUE(_LARGE_FILES, 1,
-       ac_cv_sys_large_files,
-       [Define for large files, on AIX-style hosts.],
-       AC_SYS_LARGEFILE_TEST_INCLUDES)
-     AC_SYS_LARGEFILE_MACRO_VALUE(_LARGEFILE_SOURCE, 1,
-       ac_cv_sys_largefile_source,
-       [Define to make fseeko visible on some hosts (e.g. glibc 2.2).],
-       [#include <stdio.h>], [return !fseeko;])
-   fi
-  ])
-
-
-AC_DEFUN(AC_FUNC_FSEEKO,
-  [AC_SYS_LARGEFILE_MACRO_VALUE(_LARGEFILE_SOURCE, 1,
-     ac_cv_sys_largefile_source,
-     [Define to make fseeko visible on some hosts (e.g. glibc 2.2).],
-     [#include <stdio.h>], [return !fseeko;])
-   # We used to try defining _XOPEN_SOURCE=500 too, to work around a bug
-   # in glibc 2.1.3, but that breaks too many other things.
-   # If you want fseeko and ftello with glibc, upgrade to a fixed glibc.
-
-   AC_CACHE_CHECK([for fseeko], ac_cv_func_fseeko,
-     [ac_cv_func_fseeko=no
-      AC_TRY_LINK([#include <stdio.h>],
-        [return fseeko && fseeko (stdin, 0, 0);],
-	[ac_cv_func_fseeko=yes])])
-   if test $ac_cv_func_fseeko != no; then
-     AC_DEFINE(HAVE_FSEEKO, 1,
-       [Define if fseeko (and presumably ftello) exists and is declared.])
-   fi])
-
-
 dnl XXXXXXXXXXXXXXXXXX End Stolen (but modified) from GNU tar XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-AC_DEFUN(AC_HAVE_LARGEFILES,
+AC_DEFUN([AC_HAVE_LARGEFILES],
 [AC_CACHE_CHECK([if system supports Large Files at all], ac_cv_largefiles,
      	[AC_TRY_COMPILE([#include <stdio.h>
 #include <sys/types.h>],
@@ -1492,7 +1397,7 @@
 dnl Checks for whether fseeko() is available in non large file mode
 dnl and whether there is a prototype for fseeko()
 dnl Defines HAVE_FSEEKO on success.
-AC_DEFUN(AC_SMALL_FSEEKO,
+AC_DEFUN([AC_SMALL_FSEEKO],
 [AC_CACHE_CHECK([for fseeko()], ac_cv_func_fseeko,
                 [AC_TRY_LINK([#include <stdio.h>],
 [return !fseeko;],
@@ -1505,7 +1410,7 @@
 dnl Checks for whether ftello() is available in non large file mode
 dnl and whether there is a prototype for ftello()
 dnl Defines HAVE_FTELLO on success.
-AC_DEFUN(AC_SMALL_FTELLO,
+AC_DEFUN([AC_SMALL_FTELLO],
 [AC_CACHE_CHECK([for ftello()], ac_cv_func_ftello,
                 [AC_TRY_LINK([#include <stdio.h>],
 [return !ftello;],
