--- cdrtools-2.01.01/DEFAULTS/Defaults.linux.orig	2009-06-04 23:10:45.000000000 +0200
+++ cdrtools-2.01.01/DEFAULTS/Defaults.linux	2010-01-05 09:18:52.000000000 +0100
@@ -40,16 +40,11 @@
 # LINUX_SRC_INCLUDE is used in RULES/*-linux-*cc*.rul
 #
 ###########################################################################
-LINUX_INCL_PATH=	/usr/src/linux/include
-
-_LINUX_SRC_INCLUDE=	$(_UNIQ)$(MAY_LINUX_SRC_INCLUDE)
-__LINUX_SRC_INCLUDE=	$(_LINUX_SRC_INCLUDE:$(_UNIQ)=$(LINUX_INCL_PATH))
-LINUX_SRC_INCLUDE=      $(__LINUX_SRC_INCLUDE:$(_UNIQ)no%=%)
+LINUX_SRC_INCLUDE=
 
 DEFINCDIRS=	$(SRCROOT)/include
-LDPATH=		-L/opt/schily/lib
-#RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib -R$(OLIBSDIR)
-RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib
+LDPATH=
+RUNPATH=
 
 ###########################################################################
 #
--- cdrtools-2.01.01/RULES/rules.cnf.orig	2008-08-01 20:04:08.000000000 +0200
+++ cdrtools-2.01.01/RULES/rules.cnf	2008-10-06 22:18:00.000000000 +0200
@@ -61,7 +61,7 @@
 		cd $(OINCSDIR); \
 		CONFIG_NOFAIL=TRUE CC="$(XCC_COM)" CFLAGS="$(CFLAGS)" \
 		CXXFLAGS="$(C++FLAGS)" CPPFLAGS="$(OSDEFS)" \
-		LDFLAGS="$(LDOPTS)" sh ../../autoconf/configure $(CONFFLAGS)
+		LDFLAGS="$(LDFLAGS1) $(LDOPTS)" sh ../../autoconf/configure $(CONFFLAGS)
 ###########################################################################
 include $(OINCSDIR)/Inull
 include $(OINCSDIR)/rules.cnf
